--- Scriptvdr-2.2.0_V.5.5	2015-05-12 14:33:20.169097625 +0200
+++ Script-VDR/installer	2015-05-12 15:09:07.000000000 +0200
@@ -1,7 +1,7 @@
 #!/bin/bash
 ############################## !!! IMPORTANTE !!! #######################################
 #                                                                                       #
-# Nome Script : Scriptvdr-2.2.0_V.5.5                                                   #
+# Nome Script : Scriptvdr-2.2.0_V.5.4                                                   #
 #                                                                                       #
 # Aggiornato da Knap -  ultima modifica Aprile 2015                                     #
 # Aggiornato da fiveten_59 -  ultima modifica febbraio 2015                             #
@@ -31,8 +31,8 @@
 # Per Configurazioni diverse questo Script puo' risultare non funzionante !             #
 #                                                                                       #
 # Da terminale:                                                                         #
-# date il comando sudo chmod 755 Scriptvdr-2.2.0_V.5.5                                  #
-# avviate lo script con il comando: sudo sh ./Scriptvdr-2.2.0_V.5.5                     #
+# date il comando sudo chmod 755 Scriptvdr-2.2.0_V.5.4                                  #
+# avviate lo script con il comando: sudo sh ./Scriptvdr-2.2.0_V.5.4                     #
 #                                                                                       #
 #########################################################################################
 #
@@ -262,7 +262,7 @@
     mkdir $SRC/packages $SRC/archives $SRC/scripts $SRC/READ_THIS
     while :
     do
-    dialog --title " VDR Complete Installer V.5.5 " --infobox "Copyright (C) 2010
+    dialog --title " VDR Complete Installer V.5.4 " --infobox "Copyright (C) 2010
                                                                         
     This file is part of VDR Complete Installer (VCI).                   
                                                                         
@@ -300,10 +300,10 @@
     
     #
     #    
-    dialog --title " VDR Complete Installer V.5.5 " --infobox "Copyright (C) 2010
+    dialog --title " VDR Complete Installer V.5.4 " --infobox "Copyright (C) 2010
     
 
-    Nome Script : Scriptvdr-2.2.0_V.5.5 
+    Nome Script : Scriptvdr-2.2.0_V.5.4 
 
     Aggiornato da Knap -  ultima modifica Aprile 2015                              
     Aggiornato da fiveten_59 -  ultima modifica febbraio 2015
@@ -334,8 +334,8 @@
     Per Configurazioni diverse questo Script puo' risultare non funzionante !             
                                                                                       
     Da terminale:                                                                         
-    date il comando sudo chmod 755 Scriptvdr-2.2.0_V.5.5                                
-    avviate lo script con il comando: sudo sh ./Scriptvdr-2.2.0_V.5.5                    
+    date il comando sudo chmod 755 Scriptvdr-2.2.0_V.5.4                                
+    avviate lo script con il comando: sudo sh ./Scriptvdr-2.2.0_V.5.4                    
                                                                                        
     
     =====================================                     " 0 0
@@ -344,7 +344,7 @@
     #
     clear
     echo ""
-    dialog --title "   ♦♦♦♦♦  Script Installazione VDR 2.2.0 V.5.5 - Librerie & Codec  ♦♦♦♦♦   " --infobox "
+    dialog --title "   ♦♦♦♦♦  Script Installazione VDR 2.2.0 V.5.4 - Librerie & Codec  ♦♦♦♦♦   " --infobox "
           ╭=========================================================================================╮    
           |  ----- Installa - Crea Directory ------  |  ------ Disinstalla - Rimuove Directory ---- |
           |=========================================================================================|
@@ -588,7 +588,7 @@
     wget https://www.dropbox.com/s/qa0674d9l9mgzzf/patch_softhddevice.tar.gz;
   fi
 
-if [ -s /home/`logname`/sifinstallrepo/vdr-plugin-streamdev-f1dbdd0.tar.gz ]; then
+if [ -s /home/`logname`/sifinstallrepo/vdr-plugin-streamdev ]; then
   echo "░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦::::  File Presente ... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░";
 else
   echo "░░░░░░¦¦¦¦¦¦::::  File non Presente ... Eseguo il Download ... ::::¦¦¦¦¦¦░░░░░░";
@@ -1267,6 +1267,7 @@
     git clone http://projects.vdr-developer.org/git/vdr-plugin-skinenigmang.git;
     git clone http://projects.vdr-developer.org/git/skin-flatplus.git;
     git clone https://github.com/FrodoVDR/blackholefrodo.git;
+    git clone https://github.com/Vectra130/skindesigner_vectraskin.git; mv skindesigner_vectraskin vectraskin;
     wget https://www.dropbox.com/s/odv30s0a22mau60/skinenigmang-logos-xpm-hi-20070702.tgz;
     wget https://www.dropbox.com/s/weh6c53bg9363ey/skinenigmang-channellogos-xpm-hi-20070702.tgz;
     wget https://www.dropbox.com/s/pydva1r0iafh5hy/skinenigmang-fonts-20080225.tgz;
@@ -1609,10 +1610,6 @@
 
 
     apt-get remove -y libav-tools libvpx-dev
-    apt-get update
-    apt-get install -y build-essential checkinstall git libfaac-dev libgpac-dev libjack-jackd2-dev libmp3lame-dev libopencore-amrnb-dev libopencore-amrwb-dev librtmp-dev libsdl1.2-dev
-    apt-get install -y libtheora-dev libva-dev libvdpau-dev libvorbis-dev libx11-dev libxfixes-dev pkg-config texi2html yasm zlib1g-dev
-    apt-get install -y mercurial cvs subversion git-core libncurses-dev autoconf autopoint libtool automake  gettext nasm liba52-0.7.4-dev libpng12-dev libxt-dev libxv-dev libxvmc-dev libasound2-dev libmpcdec-dev libxvidcore-dev
 
     clear
     echo "                    Adesso Installo Libvpx 1.3.0"; sleep 4
@@ -3988,7 +3985,7 @@
 read -n1 -p "Sei proprio sicuro di rimuovere Vdr? (y/n) "
 echo 
 [[ $REPLY = [yY] ]] &&
-( dpkg -r vdr; rm -r /etc/vdr; rm -r /usr/lib/vdr; cd $SRC; rm -r vdr vdr-*; cd packages; rm -r vdr*.deb; rm -r /usr/local/bin/vdrpoweroff; rm -r /usr/local/bin/vdrrun;  rm -rf /usr/local/bin/start-xine-vdr.sh; rm -rf /usr/local/bin/start-xinelib-vdr.sh; rm -rf /usr/local/bin/start-softhddevice-vdr.sh; rm -rf /home/`logname`/Scrivania/Xinelib.desktop; rm -rf /home/`logname`/Scrivania/Vdr-Softhddevice.desktop; rm -rf /home/`logname`/Scrivania/KillVdr.desktop; cd $PACKAGES; rm -rf vdr*.deb ) || echo "Hai fatto cosa buona e giusta"
+( dpkg -r vdr; rm -r /etc/vdr; rm -r /usr/lib/vdr; cd $SRC; rm -r vdr vdr-*; cd packages; rm -r vdr*.deb; rm -r /usr/local/bin/vdrpoweroff; rm -r /usr/local/bin/vdrrun;  rm -rf /usr/local/bin/start-xine-vdr.sh; rm -rf /usr/local/bin/start-xinelib-vdr.sh; rm -rf /usr/share/vdr; rm -rf /usr/local/bin/start-softhddevice-vdr.sh; rm -rf /home/`logname`/Scrivania/Xinelib.desktop; rm -rf /home/`logname`/Scrivania/Vdr-Softhddevice.desktop; rm -rf /home/`logname`/Scrivania/KillVdr.desktop; cd $PACKAGES; rm -rf vdr*.deb ) || echo "Hai fatto cosa buona e giusta"
 #
 echo ""; echo "*** Rimuovere KODI ***"; echo ""
 read -n1 -p "Sei proprio sicuro di rimuovere KODI? (y/n) "
