--- installer	2016-04-27 18:39:31.511579000 +0200
+++ installer.new	2016-04-27 19:01:00.757413000 +0200
@@ -42,9 +42,9 @@
 #
 Vdr_Xine1='sudo /etc/init.d/vdr start > /dev/null 2>&1; sleep 2; /usr/local/bin/xine -f --no-splash --no-gui -V xv --post vdr_video --post tvtime --post vdr_audio "vdr:/tmp/vdr-xine/stream#demux:mpeg_pes"'
 #
-PLUGIN="--lirc -P'xineliboutput -l none -r 37890 -p' -P'dvbapi' -P'text2skin' -P'osdteletext' -P'epgsearch'"
+PLUGIN="--lirc -P'xineliboutput -l none -r 37890 -p' -P'dvbapi' -P'skindesigner -l /etc/vdr/logos/ -i /usr/local/share/vdr/plugins/skindesigner/skins/' -P'skinflatplus -l /etc/vdr/logos/' -P'tvguideng' -P'weatherforecast' -P'iptv' -P'tvguide -l /etc/vdr/logos/'  -P'text2skin' -P'osdteletext' -P'epgsearch' -P'externalplayer'"
 # 
-PLUGIN1="--lirc -P'softhddevice -f' -P'dvbapi' -P'skindesigner -l /etc/vdr/logos/ -i /usr/share/vdr/plugins/skindesigner/skins/' -P'skinflatplus -l /etc/vdr/logos/' -P'tvguideng' -P'weatherforecast' -P'iptv' -P'tvguide -l /etc/vdr/logos/'  -P'text2skin' -P'osdteletext' -P'epgsearch' -P'externalplayer'"
+PLUGIN1="--lirc -P'softhddevice -f' -P'dvbapi' -P'skindesigner -l /etc/vdr/logos/ -i /usr/local/share/vdr/plugins/skindesigner/skins/' -P'skinflatplus -l /etc/vdr/logos/' -P'tvguideng' -P'weatherforecast' -P'iptv' -P'tvguide -l /etc/vdr/logos/'  -P'text2skin' -P'osdteletext' -P'epgsearch' -P'externalplayer'"
 #
 Remind='è già installato... avete dimenticato di aggiungere il comando al vdrrun?'
 Remind1='è già installata... avete dimenticato di aggiungere il comando al vdrrun?'
@@ -998,7 +998,7 @@
 else
   echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦:::: ..... File non Presente ... Eseguo il Download ..... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░░░░░░░░░░";
       #git clone https://github.com/maniacgit/vdr-play
-      git clone http://projects.vdr-developer.org/git/vdr-plugin-play.git/
+      git clone http://projects.vdr-developer.org/git/vdr-plugin-play.git vdr-plugin-play;
 fi
 
 if [ -s /home/`logname`/sifinstallrepo/patch_play.tar.gz ]; then
@@ -1136,7 +1136,7 @@
     wget https://www.dropbox.com/s/5l69u5htqsge852/elchiHD-logos-v3.tar.bz2;
   fi
 
-if [ -s /home/`logname`/sifinstallrepo/skinflatplus ]; then
+if [ -s /home/`logname`/sifinstallrepo/skin-flatplus ]; then
   echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦:::: ..... File Presente ..... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░";
 else
   echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦:::: ..... File non Presente ... Eseguo il Download ..... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░░░░░░░░░░";
@@ -2663,8 +2663,8 @@
     cd /usr/local/src/vdr
     apt-get -y purge vdr
     checkinstall --fstrans=no --install=yes --pkgname=vdr --pkgversion "2.2.0-git" --pakdir=$PACKAGES --default
-    rm -r /media/video; mkdir /media/video; mkdir -p /usr/lib/vdr/scripts;
-    mkdir -p /etc/vdr/plugins; cp *.conf /etc/vdr; mkdir -p /usr/lib/vdr/plugins;
+    rm -r /media/video; mkdir /media/video; mkdir -p /usr/local/lib/vdr/scripts;
+    mkdir -p /etc/vdr/plugins; cp *.conf /etc/vdr; mkdir -p /usr/local/lib/vdr/plugins;
     clear    
     echo "*** keymacros.conf setup ***";
     rm /etc/vdr/keymacros.conf; cd $SRCVDR; cp vdr /usr/local/bin/; touch /etc/vdr/keymacros.conf
@@ -2701,7 +2701,7 @@
 echo "Ora installo i Plugins base (Text2skin - OSDTeletext - Dvbapi - Softhddevice - Epgsearch - Skindesigner - TvGuideng - Weatherforecast - Skinflatplus - TvGuide - Iptv - ExternalPlayer)"
 echo "                 premere il tasto [Invio] per continuare "; read enterKey
 cd $SRC;
-mkdir -p /usr/lib/vdr/plugins; mkdir -p /usr/lib/vdr/scripts; mkdir -p /etc/vdr/plugins; mkdir -p /etc/vdr/themes; cd $PLUGINS; mkdir -p tvguide;
+mkdir -p /usr/local/lib/vdr/plugins; mkdir -p /usr/local/lib/vdr/scripts; mkdir -p /etc/vdr/plugins; mkdir -p /etc/vdr/themes; cd $PLUGINS; mkdir -p tvguide;
 #
 #
 clear
@@ -2750,6 +2750,11 @@
 cd $VDRSRC;
 cp -r /home/`logname`/sifinstallrepo/vdr-plugin-softhddevice /$VDRSRC; mv vdr-plugin-softhddevice softhddevice; cd softhddevice; cp -r /home/`logname`/sifinstallrepo/patch_softhddevice_2.tar.gz /$VDRSRC/softhddevice; tar -zxvf patch_softhddevice_2.tar.gz; rm -r patch_softhddevice_2.tar.gz; patch -p1 < 10_softhddevice_trusty.patch; sleep 1; patch -p1 < 11_Makefile.patch; sleep 1; patch -p1 < 12_ffmpeg2.2.2.patch; sleep 1; patch -p1 < 13_vdpauqueue.patch; sleep 1; patch -p1 < 15_fsigned_char.patch; sleep 1; patch -p1 < 20_ffmpeg_2.9.patch; sleep 1; make ALSA=1 OSS=0 VDPAU=1 VAAPI=0 SCREENSAVER=0 SWRESAMPLE=1; make install; 
 # 
+#clear
+#echo ""; echo "               *** Softhddevice-openglosd ***"; echo ""; sleep 3
+#cd $VDRSRC; sudo apt-get install libglew1.10 libglew-dev libglm-dev
+#cp -r /home/`logname`/sifinstallrepo/softhddevice-openglosd /$VDRSRC; cd softhddevice-openglosd; make && make install
+#  ALSA=1 OSS=0 VDPAU=1 VAAPI=0 SCREENSAVER=0 SWRESAMPLE=1;
 #
 clear
 echo ""; echo "               *** Epgsearch ***"; echo ""; sleep 3
@@ -2765,7 +2770,7 @@
 clear
 echo ""; echo "               *** Skindesigner ***"; echo ""; sleep 3
 cd $VDRSRC; sudo apt-get -y install lm-sensors librsvg2-dev libxml2-dev libcairo2-dev;
-cp -r /home/`logname`/sifinstallrepo/vdr-plugin-skindesigner /$VDRSRC/ && mv vdr-plugin-skindesigner skindesigner && cp -rf /$SRC/traduzioni/metrixhd/*.xml /$VDRSRC/skindesigner/skins/metrixhd/ && cd .. && make install && cd /usr/lib/vdr/plugins/skindesigner/scripts/ && ln -s temperatures.default temperatures; 
+cp -r /home/`logname`/sifinstallrepo/vdr-plugin-skindesigner /$VDRSRC/ && mv vdr-plugin-skindesigner skindesigner && cp -rf /$SRC/traduzioni/metrixhd/*.xml /$VDRSRC/skindesigner/skins/metrixhd/ && cd skindesigner && make install && cd /usr/local/lib/vdr/plugins/skindesigner/scripts/ && ln -s temperatures.default temperatures && ln -s vdrstats.default vdrstats; 
 # qui sopra c'è da mettere le icone di menù mancanti che sono dentro le skin del vdr-plugin-skindesigner traducendole e comprimendole
 #
 clear
@@ -2781,7 +2786,7 @@
 clear
 echo "               *** Skin Flatplus ***"; echo ""; sleep 3
 cd $VDRSRC;
-cp -r /home/`logname`/sifinstallrepo/skin-flatplus /$VDRSRC/ && cd skin-flatplus && make install && cd /usr/lib/vdr/plugins/skinflatplus/widgets/temperatures/ && ln -s temperatures.default temperatures && cd ../system_information && ln -s system_information.ubuntu system_information  && cp -rf /home/`logname`/sifinstallrepo/flat_weather.tar.gz ../weather/ && cd ../weather && tar -zxvf flat_weather.tar.gz && rm -r flat_weather.tar.gz;
+cp -r /home/`logname`/sifinstallrepo/skin-flatplus /$VDRSRC/ && cd skin-flatplus && make install && cd /usr/local/lib/vdr/plugins/skinflatplus/widgets/temperatures/ && ln -s temperatures.default temperatures && cd ../system_information && ln -s system_information.ubuntu system_information  && cp -rf /home/`logname`/sifinstallrepo/flat_weather.tar.gz ../weather/ && cd ../weather && tar -zxvf flat_weather.tar.gz && rm -r flat_weather.tar.gz;
 # qui sopra c'è da mettere le icone di menù mancanti che sono dentro le skin del Skin Flatplus traducendole e comprimendole
 #
 clear
@@ -2790,7 +2795,7 @@
 #
 #
 echo "               *** Iptv ***"; echo ""; sleep 3
-cd $VDRSRC && cp -r /home/`logname`/sifinstallrepo/vdr-iptv-2.2.1.tgz /$VDRSRC && tar -zxvf vdr-iptv-2.2.1.tgz && mv iptv-2.2.1 iptv && rm -r vdr-iptv-2.2.1.tgz && cd iptv && cp -r /home/`logname`/sifinstallrepo/patch_iptv.tar.gz /$VDRSRC/iptv && tar -zxvf patch_iptv.tar.gz && rm -r patch_iptv.tar.gz && patch -p1 < iptv.patch && make && cd ../../.. && make plugins && mkdir -p /etc/vdr/plugins/iptv/vlcinput && cp -r /home/`logname`/sifinstallrepo/iptv_conf.tar.gz /$PLUGINS/iptv/vlcinput/ && cd $PLUGINS/iptv/vlcinput && tar -zxvf iptv_conf.tar.gz && rm -r iptv_conf.tar.gz 
+cd $VDRSRC && apt-get -y install vlc && cp -r /home/`logname`/sifinstallrepo/vdr-iptv-2.2.1.tgz /$VDRSRC && tar -zxvf vdr-iptv-2.2.1.tgz && mv iptv-2.2.1 iptv && rm -r vdr-iptv-2.2.1.tgz && cd iptv && cp -r /home/`logname`/sifinstallrepo/patch_iptv.tar.gz /$VDRSRC/iptv && tar -zxvf patch_iptv.tar.gz && rm -r patch_iptv.tar.gz && patch -p1 < iptv.patch && make && cd ../../.. && make plugins && mkdir -p /etc/vdr/plugins/iptv/vlcinput && cp -r /home/`logname`/sifinstallrepo/iptv_conf.tar.gz /$PLUGINS/iptv/vlcinput/ && cd $PLUGINS/iptv/vlcinput && tar -zxvf iptv_conf.tar.gz && rm -r iptv_conf.tar.gz 
 #
 #
 clear
@@ -2818,7 +2823,7 @@
 
     clear
     echo "                    Adesso Rimuovo VDR 2.2.0"; sleep 4
-    apt-get -y purge vdr; rm /usr/local/src/vdr; rm -r /usr/local/src/vdr-2.2.0; rm /usr/local/src/packages/vdr*; rm -r /usr/lib/vdr; rm /usr/share/locale/ca_ES/LC_MESSAGES;
+    apt-get -y purge vdr; rm /usr/local/src/vdr; rm -r /usr/local/src/vdr-2.2.0; rm /usr/local/src/packages/vdr*; rm -r /usr/local/lib/vdr; rm /usr/share/locale/ca_ES/LC_MESSAGES;
     echo "                    Premere il tasto [Invio] per continuare. . .               "; read enterKey;;
 
     
@@ -4524,7 +4529,7 @@
     echo -n "        Scegli  [a-b]:      "  
     read yourch
     case $yourch in
-a) cd /usr/share/vdr/plugins/skindesigner/skins; git clone https://github.com/BooStars/stillClassic.git;
+a) cd /usr/local/share/vdr/plugins/skindesigner/skins; git clone https://github.com/BooStars/stillClassic.git;
 find -name *.theme -exec mv {} "/etc/vdr/themes" \ ;
       echo "                    Premere il tasto [Invio] per continuare. . .               "; read enterKey;;
 b) echo "Avete scelto di non installare altre skin di Skindesigner."
@@ -4544,7 +4549,7 @@
     echo -n "        Scegli  [a-b]:      "  
     read yourch
     case $yourch in
-a) cd /usr/share/vdr/plugins/skindesigner/skins/ && if [ -f blackhole/globals.xml ];
+a) cd /usr/local/share/vdr/plugins/skindesigner/skins/ && if [ -f blackhole/globals.xml ];
   then  cp -Rf /$SRC/traduzioni/blackhole/*.xml $SKINS/blackhole/ 
   	else echo "$Remind8"
 	fi
@@ -4739,8 +4744,8 @@
     echo -n "        Scegli  [a-g]:      "                   
     read yourch
     case $yourch in
-    a) rm -r $VDRSRC/skindesigner; rm -rf /usr/share/vdr/plugins/skindesigner; rm -rf /usr/lib/vdr/plugins/skindesigner; rm -r $PLUGINS/skindesigner;;
-    b) rm -r $VDRSRC/skin-flatplus; rm -r $PLUGINS/skinflatplus; rm -rf /usr/lib/vdr/plugins/skinflatplus; rm -rf /usr/share/vdr/plugins/skinflatplus;;
+    a) rm -r $VDRSRC/skindesigner; rm -rf /usr/local/share/vdr/plugins/skindesigner; rm -rf /usr/local/lib/vdr/plugins/skindesigner; rm -r $PLUGINS/skindesigner;;
+    b) rm -r $VDRSRC/skin-flatplus; rm -r $PLUGINS/skinflatplus; rm -rf /usr/local/lib/vdr/plugins/skinflatplus; rm -rf /usr/local/share/vdr/plugins/skinflatplus;;
     c) rm -r $VDRSRC/narrowhd; rm -r $SKIN/NarrowHD;;
     d) rm -r $VDRSRC/skinenigmang;;
     e) rm -r $VDRSRC/skinsoppalusikka; rm -r /etc/vdr/plugins/skinsoppalusikka;;
@@ -4798,8 +4803,8 @@
     echo -n "        Scegli  [a-c]:      "
     read yourch
     case $yourch in
-a) cd $VDRSRC && rm -rf skindesigner && rm -rf /usr/lib/vdr/plugins/skindesigner && rm -rf /usr/share/vdr/plugins/skindesigner && rm -rf /home/`logname`/sifinstallrepo/vdr-plugin-skindesigner && cd $PLUGINS && rm -rf skindesigner && cd $VDRSRC && git clone git://projects.vdr-developer.org/vdr-plugin-skindesigner.git && cp -rf vdr-plugin-skindesigner /home/`logname`/sifinstallrepo/  && mv vdr-plugin-skindesigner skindesigner && cd skindesigner && apt-get -y install lm-sensors librsvg2-dev libxml2-dev libcairo2-dev && make install && cd /usr/lib/vdr/plugins/skindesigner/scripts/ && ln -s temperatures.default temperatures ;;
-b) cd $VDRSRC && rm -rf skindesigner && rm -rf /usr/share/vdr/plugins/skindesigner && cd $PLUGINS && rm -rf skindesigner && sed -i "s|-P'skindesigner -l /etc/vdr/logos/ -i /usr/share/vdr/plugins/skindesigner/skins/'||" /usr/local/bin/vdrrun;;
+a) cd $VDRSRC && rm -rf skindesigner && rm -rf /usr/local/lib/vdr/plugins/skindesigner && rm -rf /usr/local/share/vdr/plugins/skindesigner && rm -rf /home/`logname`/sifinstallrepo/vdr-plugin-skindesigner && cd $PLUGINS && rm -rf skindesigner && cd $VDRSRC && git clone git://projects.vdr-developer.org/vdr-plugin-skindesigner.git && cp -rf vdr-plugin-skindesigner /home/`logname`/sifinstallrepo/  && mv vdr-plugin-skindesigner skindesigner && cd skindesigner && apt-get -y install lm-sensors librsvg2-dev libxml2-dev libcairo2-dev && make install && cd /usr/local/lib/vdr/plugins/skindesigner/scripts/ && ln -s temperatures.default temperatures && ln -s vdrstats.default vdrstats;;
+b) cd $VDRSRC && rm -rf skindesigner && rm -rf /usr/local/share/vdr/plugins/skindesigner && rm -rf /usr/local/share/vdr/plugins/skindesigner && cd $PLUGINS && rm -rf skindesigner && sed -i "s|-P'skindesigner -l /etc/vdr/logos/ -i /usr/local/share/vdr/plugins/skindesigner/skins/'||" /usr/local/bin/vdrrun;;
 c) ;;
 esac
 #
@@ -4878,8 +4883,8 @@
     echo -n "        Scegli  [a-c]:      "
     read yourch
     case $yourch in
-a) cd $VDRSRC && rm -rf skin-flatplus rm -rf /home/`logname`/sifinstallrepo/skin-flatplus && rm -rf /usr/lib/vdr/plugins/skinflatplus && rm -rf /usr/share/vdr/plugins/skinflatplus && cd $PLUGINS && rm -rf skinflatplus && cd $VDRSRC && git clone http://projects.vdr-developer.org/git/skin-flatplus.git/ && cp -rf skin-flatplus /home/`logname`/sifinstallrepo/ && cd skin-flatplus && make install && cd /usr/lib/vdr/plugins/skinflatplus/widgets/temperatures/ && ln -s temperatures.default temperatures && cd ../system_information && ln -s system_information.ubuntu system_information  && cp -rf /home/`logname`/sifinstallrepo/flat_weather.tar.gz ../weather/ && cd ../weather && tar -zxvf flat_weather.tar.gz && rm -r flat_weather.tar.gz ;;
-b) cd $VDRSRC && rm -rf skin-flatplus && cd $PLUGINS && rm -rf skinflatplus && rm -rf /usr/lib/vdr/plugins/skinflatplus && rm -rf /usr/share/vdr/plugins/skinflatplus && sed -i "s|-P'skinflatplus -l /etc/vdr/logos/'||" /usr/local/bin/vdrrun;;
+a) cd $VDRSRC && rm -rf skin-flatplus rm -rf /home/`logname`/sifinstallrepo/skin-flatplus && rm -rf /usr/local/lib/vdr/plugins/skinflatplus && rm -rf /usr/local/share/vdr/plugins/skinflatplus && cd $PLUGINS && rm -rf skinflatplus && cd $VDRSRC && git clone http://projects.vdr-developer.org/git/skin-flatplus.git/ && cp -rf skin-flatplus /home/`logname`/sifinstallrepo/ && cd skin-flatplus && make install && cd /usr/local/lib/vdr/plugins/skinflatplus/widgets/temperatures/ && ln -s temperatures.default temperatures && cd ../system_information && ln -s system_information.ubuntu system_information  && cp -rf /home/`logname`/sifinstallrepo/flat_weather.tar.gz ../weather/ && cd ../weather && tar -zxvf flat_weather.tar.gz && rm -r flat_weather.tar.gz ;;
+b) cd $VDRSRC && rm -rf skin-flatplus && cd $PLUGINS && rm -rf skinflatplus && rm -rf /usr/local/lib/vdr/plugins/skinflatplus && rm -rf /usr/local/share/vdr/plugins/skinflatplus && sed -i "s|-P'skinflatplus -l /etc/vdr/logos/'||" /usr/local/bin/vdrrun;;
 c) ;;
 esac
 #
@@ -4998,8 +5003,8 @@
     echo -n "        Scegli  [a-c]:      "
     read yourch
     case $yourch in 
-a) cd $VDRSRC && rm -r iptv && rm -rf /etc/vdr/plugins/iptv && rm -r /usr/share/vdr/plugins/iptv && cp -r /home/`logname`/sifinstallrepo/vdr-iptv-2.2.1.tgz /$VDRSRC && tar -zxvf vdr-iptv-2.2.1.tgz && mv iptv-2.2.1 iptv && rm -r vdr-iptv-2.2.1.tgz && cd iptv && cp -r /home/`logname`/sifinstallrepo/patch_iptv.tar.gz /$VDRSRC/iptv && tar -zxvf patch_iptv.tar.gz && rm -r patch_iptv.tar.gz && patch -p1 < iptv.patch && make && cd ../../.. && make plugins && mkdir -p /etc/vdr/plugins/iptv/vlcinput && cp -r /home/`logname`/sifinstallrepo/iptv_conf.tar.gz /$PLUGINS/iptv/vlcinput/ && cd $PLUGINS/iptv/vlcinput && tar -zxvf iptv_conf.tar.gz && rm -r iptv_conf.tar.gz;;
-b) cd $VDRSRC && rm -r iptv && rm -rf $PLUGINS/iptv && rm -r /usr/share/vdr/plugins/iptv && sed -i "s|-P'iptv'||" /usr/local/bin/vdrrun;;
+a) cd $VDRSRC && rm -r iptv && rm -rf /etc/vdr/plugins/iptv && rm -r /usr/local/share/vdr/plugins/iptv && cp -r /home/`logname`/sifinstallrepo/vdr-iptv-2.2.1.tgz /$VDRSRC && tar -zxvf vdr-iptv-2.2.1.tgz && mv iptv-2.2.1 iptv && rm -r vdr-iptv-2.2.1.tgz && cd iptv && cp -r /home/`logname`/sifinstallrepo/patch_iptv.tar.gz /$VDRSRC/iptv && tar -zxvf patch_iptv.tar.gz && rm -r patch_iptv.tar.gz && patch -p1 < iptv.patch && make && cd ../../.. && make plugins && mkdir -p /etc/vdr/plugins/iptv/vlcinput && cp -r /home/`logname`/sifinstallrepo/iptv_conf.tar.gz /$PLUGINS/iptv/vlcinput/ && cd $PLUGINS/iptv/vlcinput && tar -zxvf iptv_conf.tar.gz && rm -r iptv_conf.tar.gz;;
+b) cd $VDRSRC && rm -r iptv && rm -rf $PLUGINS/iptv && rm -r /usr/local/share/vdr/plugins/iptv && sed -i "s|-P'iptv'||" /usr/local/bin/vdrrun;;
 c) ;;
 esac
 #
@@ -5299,15 +5304,15 @@
     ###  Sezione Vdr-Getchannels 0.7.0 Old  ###
     ###
 
-    x1) cd $SRC; clear; rm -rf settings; mkdir settings; clear;
-    cd settings; cp -r /home/`logname`/sifinstallrepo/getchannels.tar.gz $SRC/settings;
-    tar -zxvf getchannels.tar.gz; cd /usr/local/src/settings/getchannels/; 
-    python getchannels.py  --list 13.0E --output output/channels13.conf; sleep  4; cp $SRC/setting/getchannels/output/channels13.conf /etc/vdr/;;
-
-    x2) cd $SRC; clear; rm -rf settings; mkdir settings; clear;
-    cd settings; cp -r /home/`logname`/sifinstallrepo/getchannels.tar.gz $SRC/settings;
-    tar -zxvf getchannels.tar.gz; cd /usr/local/src/settings/getchannels/;
-    python getchannels.py  --list 19.2E --output output/channels19.conf; sleep  4; cp $SRC/setting/getchannels/output/channels19.conf /etc/vdr/;;
+    x1) apt-get -y install unzip; clear; 
+    cd $SRC; rm -rf setting; rm -rf getchannels; mkdir setting; cd setting; mkdir e2setting;
+    cp -r /home/`logname`/sifinstallrepo/getchannels.tar.gz $SRC/setting; tar -zxvf getchannels.tar.gz; #rm -r getchannels.tar.gz; 
+    cd $SRC/setting/getchannels && python getchannels.py  --list 13.0E --output output/channels13.conf; sleep  4; cp $SRC/setting/getchannels/output/channels13.conf /etc/vdr/;;
+
+    x2) apt-get -y install unzip; clear; 
+    cd $SRC; rm -rf setting; rm -rf getchannels; mkdir setting; cd setting; mkdir e2setting;
+    cp -r /home/`logname`/sifinstallrepo/getchannels.tar.gz $SRC/setting; tar -zxvf getchannels.tar.gz; #rm -r getchannels.tar.gz; 
+    cd $SRC/setting/getchannels && python getchannels.py  --list 19.2E --output output/channels19.conf; sleep  4; cp $SRC/setting/getchannels/output/channels19.conf /etc/vdr/;;
   
     x3) cp -rf /home/`logname`/sifinstallrepo/channels.conf.trial.tar.gz /etc/vdr/;;
    
@@ -6032,7 +6037,7 @@
     echo -n "        Scegli  [a-b]:      "
     read yourch
     case $yourch in
-a)  dpkg -r vdr; rm -rf /etc/vdr; rm -rf /usr/share/vdr; rm -rf /usr/lib/vdr; cd $SRC; rm -r vdr vdr-*; cd packages; rm -r vdr*.deb; rm -r /usr/local/bin/vdrpoweroff; rm -r /usr/local/bin/vdrrun;  rm -rf /usr/local/bin/start-vdr-xine.sh; rm -rf /usr/local/bin/start-xinelib-vdr.sh; rm -rf /usr/share/vdr; rm -rf /usr/local/bin/start-softhddevice-vdr.sh; rm -rf /home/`logname`/Scrivania/Xinelib.desktop; rm -rf /home/`logname`/Scrivania/Vdr-Xine.desktop; rm -rf /home/`logname`/Scrivania/Vdr-Softhddevice.desktop; rm -r /usr/bin/mplayer.sh; rm -rf /home/`logname`/Scrivania/KillVdr.desktop; cd $PACKAGES; rm -rf vdr*.deb;;
+a)  dpkg -r vdr; rm -rf /etc/vdr; rm -rf /usr/local/share/vdr; rm -rf /usr/local/lib/vdr; cd $SRC; rm -r vdr vdr-*; cd packages; rm -r vdr*.deb; rm -r /usr/local/bin/vdrpoweroff; rm -r /usr/local/bin/vdrrun;  rm -rf /usr/local/bin/start-vdr-xine.sh; rm -rf /usr/local/bin/start-xinelib-vdr.sh; rm -rf /usr/local/share/vdr; rm -rf /usr/local/bin/start-softhddevice-vdr.sh; rm -rf /home/`logname`/Scrivania/Xinelib.desktop; rm -rf /home/`logname`/Scrivania/Vdr-Xine.desktop; rm -rf /home/`logname`/Scrivania/Vdr-Softhddevice.desktop; rm -r /usr/bin/mplayer.sh; rm -rf /home/`logname`/Scrivania/KillVdr.desktop; cd $PACKAGES; rm -rf vdr*.deb;;
 b) ;;
 esac
 #
