--- Script-VDR-Scriptvdr-2.2.0_V.5.6.3/installer	2016-03-29 18:13:42.535991335 +0200
+++ Script-VDR-Scriptvdr-2.2.0_V.5.6.4/installer	2016-04-22 19:09:46.614386585 +0200
@@ -419,28 +419,36 @@
 
     clear
     dialog --title "   ** Download & Update Repository Locale **   " --infobox "
-   --------------------------------------------------------------
-                   Cartella Repository in Locale
-   --------------------------------------------------------------
+   ---------------------------------------------------------------------------------------------
+       Creazione ~ Aggiornamento ~ Rimozione ~ Cartella Repository Locale Sifinstallrepo 
+   ---------------------------------------------------------------------------------------------
         
-        *****  Crea  *****  ........  *****  Rimuove  *****
-        [a]  Creazione  ............  [a1]  Sorgenti di VDR  
-        [b]  Aggiornamento  ........  [b1]  Sorgenti di DVBAPI
-        [c]  .......................  [c1]  Sorgenti di SC  
-        [d]  .......................  [d1]  Tutto il contenuto
-
-        [e]  Skip
+        *****  Crea  ***** 
+        [a]  Creazione ~ (Se Non Presente) ............ [b]  Aggiornamento ~ (Rimuove e Aggiorna Tutto)
         
         
+        *****  Rimuove  *****
+        [c]  libdrm-2.4.x  ............................ [d]  libva-1.x 
+        [e]  libva-intel-driver-1.6.x  ................ [f]  libva-vdpau-driver-0.7.x
+        [g]  libvpx-1.x  .............................. [h]  ffmpeg-2.x.x
+        [i]  vdr-patches.2.2.x  ....................... [l]  Make.config.2.2.x
+        [m]  vdr-cecremote-x  ......................... [n]  externalplayer-x
+        [o]  Sorgenti di VDR  ......................... [p]  Sorgenti di DVBAPI
+        [q]  Sorgenti di SC  .......................... [r]  Tutto il contenuto
+        
+        [z]  Skip      
         
-   --------------------------------------------------------------
+   ---------------------------------------------------------------------------------------------
         Note:
-        - [a]  Creazione  -  Crea File Se Non Presente.
+        - [a]  Creazione  -  Scarica il File Se Non Presente.
+        - [b]  Aggiornamento  -  Funzione Tradizionale - Rimuove e Aggiorna Tutto.
         
-        - [b]  Aggiornamento  -  Funzione Tradizionale.
+        Funzione Rimuove: 
+        - Rimozione pacchetti obsoleti e versioni precedenti non più in uso o aggiornati.
+        - libera spazio nella Sifinstallrepo.
  
-   --------------------------------------------------------------" 0 0
-    echo -n "        Scegli  [a-e]:      "
+   ---------------------------------------------------------------------------------------------" 0 0
+    echo -n "        Scegli = Crea [a-b] ~~ Rimuove [c-r] ~~ Esci [z]:      "
     read yourch
     case $yourch in
     a) clear; cd /home/`logname`; mkdir -p sifinstallrepo;
@@ -461,20 +469,20 @@
 else
   echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦:::: ..... File non Presente ... Eseguo il Download ..... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░░░░░░░░░░";
     wget https://dri.freedesktop.org/libdrm/libdrm-2.4.67.tar.gz;
-  fi    
+  fi
 
-if [ -s /home/`logname`/sifinstallrepo/libva-1.6.2.tar.bz2 ]; then
+if [ -s /home/`logname`/sifinstallrepo/libva-1.7.0.tar.bz2 ]; then
   echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦:::: ..... File Presente ..... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░";
 else
   echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦:::: ..... File non Presente ... Eseguo il Download ..... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░░░░░░░░░░";
-    wget https://www.freedesktop.org/software/vaapi/releases/libva/libva-1.6.2.tar.bz2;
-  fi    
+    wget https://www.freedesktop.org/software/vaapi/releases/libva/libva-1.7.0.tar.bz2;
+  fi
 
-if [ -s /home/`logname`/sifinstallrepo/libva-intel-driver-1.6.2.tar.bz2 ]; then
+if [ -s /home/`logname`/sifinstallrepo/libva-intel-driver-1.7.0.tar.bz2 ]; then
   echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦:::: ..... File Presente ..... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░";
 else
   echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦:::: ..... File non Presente ... Eseguo il Download ..... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░░░░░░░░░░";
-    wget https://www.freedesktop.org/software/vaapi/releases/libva-intel-driver/libva-intel-driver-1.6.2.tar.bz2;
+    wget https://www.freedesktop.org/software/vaapi/releases/libva-intel-driver/libva-intel-driver-1.7.0.tar.bz2;
   fi      
 
 if [ -s /home/`logname`/sifinstallrepo/libva-vdpau-driver-0.7.4.tar.bz2 ]; then
@@ -561,6 +569,13 @@
     wget http://developer.download.nvidia.com/assets/cuda/files/nvidia_video_sdk_6.0.1.zip;
   fi
 
+if [ -s /home/`logname`/sifinstallrepo/ffmpeg-2.8.tar.bz2 ]; then
+  echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦:::: ..... File Presente ..... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░";
+else
+  echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦:::: ..... File non Presente ... Eseguo il Download ..... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░░░░░░░░░░";
+    wget https://www.ffmpeg.org/releases/ffmpeg-2.8.tar.bz2;
+  fi
+
 if [ -s /home/`logname`/sifinstallrepo/ffmpeg-2.8.6.tar.bz2 ]; then
   echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦:::: ..... File Presente ..... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░";
 else
@@ -1334,8 +1349,7 @@
     wget https://www.dropbox.com/s/v6o2zsjt7yvascy/flat_weather.tar.gz;
   fi
 
-#
-  chmod -R 777 *;;
+chmod -R 777 *;;
   
 ######################################################################################  
 
@@ -1346,17 +1360,18 @@
 
     wget https://www.dropbox.com/s/jtk3ax8hj7dmf2h/yasm-1.3.0.tar.gz;
     wget https://dri.freedesktop.org/libdrm/libdrm-2.4.67.tar.gz;
-    wget https://www.freedesktop.org/software/vaapi/releases/libva/libva-1.6.2.tar.bz2;
-    wget https://www.freedesktop.org/software/vaapi/releases/libva-intel-driver/libva-intel-driver-1.6.2.tar.bz2;
+    wget https://www.freedesktop.org/software/vaapi/releases/libva/libva-1.7.0.tar.bz2;
+    wget https://www.freedesktop.org/software/vaapi/releases/libva-intel-driver/libva-intel-driver-1.7.0.tar.bz2;
     wget http://www.freedesktop.org/software/vaapi/releases/libva-vdpau-driver/libva-vdpau-driver-0.7.4.tar.bz2;
     wget https://www.dropbox.com/s/3hbp5r2jf4a1lh8/vdpau_decode.patch;
     wget https://www.dropbox.com/s/u8h7u5htmqdzd0n/vdpau_dump.patch;
     wget https://www.dropbox.com/s/hvu4nstph14zaz5/utils_glx.patch;
     git clone https://github.com/mstorsjo/fdk-aac.git;
-    git clone git://git.videolan.org/x264.git
+    git clone git://git.videolan.org/x264.git;
     wget http://storage.googleapis.com/downloads.webmproject.org/releases/webm/libvpx-1.4.0.tar.bz2;
-    wget http://storage.googleapis.com/downloads.webmproject.org/releases/webm/libvpx-1.5.0.tar.bz2;    
+    wget http://storage.googleapis.com/downloads.webmproject.org/releases/webm/libvpx-1.5.0.tar.bz2;
     wget https://www.ffmpeg.org/releases/ffmpeg-2.8.tar.bz2;
+    wget https://github.com/FFmpeg/FFmpeg/releases/download/n2.8.6/ffmpeg-2.8.6.tar.bz2;
 
 
 ############################ caricamento xinelib - xine-ui ###########################
@@ -1495,13 +1510,22 @@
                                           #########################################
                                           # Rimozione Directory Repository Locale #
                                           #########################################
-
-   
-    a1) clear; rm /home/`logname`/sifinstallrepo/vdr-2.2.0.tar.bz2;;
-    b1) clear; rm -r /home/`logname`/sifinstallrepo/vdr-plugin-dvbapi;;
-    c1) clear; rm -r /home/`logname`/sifinstallrepo/sc;;
-    d1) clear; rm -r /home/`logname`/sifinstallrepo;;
-    e) ;;
+        
+    c) rm -rf libdrm-2.4.*;;
+    d) rm -rf libva-1.*;;
+    e) rm -rf libva-intel-driver-1.6.*;;
+    f) rm -rf libva-vdpau-driver-0.7.*;;
+    g) rm -rf libvpx-1.*;;
+    h) rm -rf ffmpeg-*;;
+    i) rm -rf vdr-patches.*;;
+    l) rm -rf Make.config.*;;
+    m) rm -rf vdr-cecremote-*;;
+    n) rm -rf externalplayer.tar.gz;;
+    o) clear; rm /home/`logname`/sifinstallrepo/vdr-2.2.0.tar.bz2;;
+    p) clear; rm -r /home/`logname`/sifinstallrepo/vdr-plugin-dvbapi;;
+    q) clear; rm -r /home/`logname`/sifinstallrepo/sc;;
+    r) clear; rm -r /home/`logname`/sifinstallrepo;;
+    z) ;;
     esac
     echo " Premere il tasto [Invio] per continuare. . . "; read enterKey;;
     
@@ -1520,44 +1544,41 @@
     clear        
     dialog --title "   ** Supporto Librerie Schede Video ATI & Intel & Nvidia - YASM - LIBDRM - LIBVA - LIBVDPAU  **   " --infobox "
    ---------------------------------------------------------------------------------------------
-            Installa o Rimuove Librerie di supporto {Vaapi-Vdpau} - {Test Softhddevice}
+            Installa o Rimuove Librerie di supporto {Vaapi ~ Vdpau} - {Softhddevice}
    ---------------------------------------------------------------------------------------------
+   
+        ***** Installa  *****  .............................  *****  Rimuove  *****
+        [a]    YASM ......... 1.3.0  ........................  [a1]    YASM ......... 1.3.0
         
-        ***** Installa  *****  .........................  *****  Rimuove  *****
-        [a]    YASM ......... 1.3.0  .....................  [a1]    YASM ......... 1.3.0
-        
-        [b]    LIBDRM ....... 2.4.67  ....................  [b1]    LIBDRM ....... 2.4.67
+        [b]    LIBDRM ....... 2.4.67  .......................  [b1]    LIBDRM ....... 2.4.67
         
-        [c]    LIBVA ........ 1.6.2  .....................  [c1]    LIBVA ........ 1.6.2
-        [d]    LIBVA-INTEL .. 1.6.2  .....................  [d1]    LIBVA-INTEL .. 1.6.2
-        [e]    LIBVA-VDPAU .. 0.7.4  .....................  [e1]    LIBVA-VDPAU .. 0.7.4
+        [c]    LIBVA ........ 1.6.2  ........................  [c1]    LIBVA ........ 1.6.2
+        [d]    LIBVA-INTEL-DRIVER .. 1.6.2  .................  [d1]    LIBVA-INTEL-DRIVER .. 1.6.2
+        [e]    LIBVA-VDPAU-DRIVER .. 0.7.4  .................  [e1]    LIBVA-VDPAU-DRIVER .. 0.7.4
         
-        [f]    LIB-VDPAU .... 1.1.1  .....................  [f1]    LIB-VDPAU .... 1.1.1
-        [g]    VDPAUINFO .... 1.0  .......................  [g1]    VDPAUINFO .... 1.0
+        [f]    LIB-VDPAU .... 1.1.1  ........................  [f1]    LIB-VDPAU .... 1.1.1
+        [g]    VDPAUINFO .... 1.0  ..........................  [g1]    VDPAUINFO .... 1.0
         
         [z]    Skip
         
    ---------------------------------------------------------------------------------------------      
-        Note:
-        - LIBVA 1.6.2 - Va sempre installato come dipendenza.
+        Note:  YASM ~ LIBDRM ~ LIBVA 1.6.2 - Vanno sempre installati come dipendenze.
         
-        Note:
-        - LIBVA-INTEL-DRIVER 1.6.2 - is the VA-API implementation for Intel G45 chipsets
+        Note:  LIBDRM ~ Mesa 3D and DRI [(Direct Rendering Infrastructure) by 3DFX, AMD (formerly ATI), Intel and Matrox]
+        
+        Note:  LIBVA ~ Fornisce l'accesso all'accelerazione grafica hardware (GPU) GMA (Graphics Media Accelerator).
+        
+        Note:  LIBVA-INTEL-DRIVER 1.6.2 - is the VA-API implementation for Intel G45 chipsets
         and Intel HD Graphics for Intel Core processor family.
 
         Platform definitions:
-        CTG: Cantiga, Intel GMA 4500MHD (GM45)
-        ILK: Ironlake, Intel HD Graphics for 2010 Intel Core processor family
-        SNB: Sandybridge, Intel HD Graphics for 2011 Intel Core processor family
-        IVB: Ivybridge
+        CTG: Cantiga, Intel GMA 4500MHD (GM45) - ILK: Ironlake, Intel HD Graphics for 2010 Intel Core processor family
+        SNB: Sandybridge, Intel HD Graphics for 2011 Intel Core processor family - IVB: Ivybridge
         
-        Note:
-        - LIBVA-VDPAU-DRIVER 0.7.4 - consists in a VDPAU-based backend for VA-API.
+        Note:  LIBVA-VDPAU-DRIVER 0.7.4 - consists in a VDPAU-based backend for VA-API.
         
-        Note:
-        - LIB-VDPAU 1.1.1 - Assegna parte del processo della decodifica e della 
+        Note:  LIB-VDPAU 1.1.1 - (Nvidia GPU hardware) Assegna parte del processo della decodifica e della 
         post-elaborazione video alla GPU della scheda video. 
-   
    --------------------------------------------------------------------------------------------- " 0 0
     echo -n "        Scegli = Installa [a-g] ~~ Rimuove [a1-g1] ~~ Esci [z]:      "
     read yourch
@@ -1903,19 +1924,23 @@
                                           #######################################################
                   
     clear        
-    dialog --title "   ** Supporto Librerie Librerie & Codec - X264 - X265 - FDK-AAC - LIBVPX - FFMPEG **   " --infobox "
+    dialog --title "   ** Supporto Librerie & Codec - X264 - X265 - FDK-AAC - LIBVPX - FFMPEG **   " --infobox "
    --------------------------------------------------------------------------------------------------------------------
                                      Installa o Rimuove Librerie di Supporto
    --------------------------------------------------------------------------------------------------------------------
         
-        *****  Installa  *****  ......................  *****  Rimuove  *****
-        [a]    X264 26.09.2015 Git  ....................  [a1]    X264 26.09.2015 Git
-        [b]    X265 HG  ................................  [b1]    X265 HG
-        [c]    FDK-AAC Git  ............................  [c1]    FDK-AAC Git
+        *****  Installa  *****  ...........................  *****  Rimuove  *****
+        [a]    X264 26.09.2015 Git  .......................  [a1]    X264 26.09.2015 Git
+        [b]    X265 HG  ...................................  [b1]    X265 HG
+        [c]    FDK-AAC Git  ...............................  [c1]    FDK-AAC Git
         
-        [d]    NVIDIA Video Codec 6.1 SDK ..............  [d1]    NVIDIA Video Codec 6.1 SDK        
-        [e]    LIBVPX 1.5.0 ............................  [e1]    LIBVPX 1.5.0
-        [f]    FFMPEG 2.8.6 Git-Snapshot ...............  [f1]    FFMPEG 2.8.6 Git-Snapshot
+        [d]    NVIDIA Video Codec 6.1 SDK .................  [d1]    NVIDIA Video Codec 6.1 SDK
+        
+        [e]    LIBVPX 1.4.0 ~ Stabile .....................  [e1]    LIBVPX 1.4.0 ~ Stabile
+        [f]    FFMPEG 2.8.0 ~ Stabile - Git-Snapshot ......  [f1]    FFMPEG 2.8.0 ~ Stabile - Git-Snapshot
+        
+        [g]    LIBVPX 1.5.0 ~ Testing .....................  [g1]    LIBVPX 1.5.0 ~ Testing
+        [h]    FFMPEG 2.8.6 ~ Testing - Git-Snapshot ......  [h1]    FFMPEG 2.8.6 ~ Testing - Git-Snapshot
         
         [z]    Skip
         
@@ -1926,12 +1951,13 @@
         ~$ sudo apt-get purge x264 libx264-dev ffmpeg
         
         Note:
-        - FFMPEG 2.8.6 Stabile - Si puo' selezionare il supporto per X264 & X265
+        - FFMPEG 2.8.0 Stabile - Si puo' selezionare il supporto per X264 & X265 - Supporto per NVENC
+        - FFMPEG 2.8.6 Testing - Si puo' selezionare il supporto per X264 & X265 - Supporto per NVENC
         
         Note:
         - NVIDIA Video Codec 6.1 SDK --- NVENC Hardware Accelerated Video Encoding Decoding ~ H.264 and H.265 (HEVC) 
    -------------------------------------------------------------------------------------------------------------------- " 0 0
-    echo -n "        Scegli = Installa [a-f] ~~ Rimuove [a1-f1] ~~ Esci [z]:      "
+    echo -n "        Scegli = Installa [a-h] ~~ Rimuove [a1-h1] ~~ Esci [z]:      "
     read yourch
     case $yourch in
     
@@ -2105,7 +2131,7 @@
 
 
     clear
-    echo "                    Adesso Rimuovo Libvpx 1.5.0"; sleep 4
+    echo "                    Adesso Rimuovo NVIDIA Video Codec 6.1 SDK"; sleep 4
     rm -r /usr/local/src/nvidia_sdk;;
 
 
@@ -2117,6 +2143,140 @@
 ##                                                                                                                         ##
 #                                                                                                                           #
 e)                                        #########################################
+                                          #              LIBVPX  1.4.0            #
+                                          #########################################
+
+
+    apt-get -y remove libvpx
+    clear
+    apt-get update
+    ### Aggiunto Supporto per Linux Mint 16 Mate ###
+    apt-get -y install build-essential checkinstall git libfaac-dev libgpac-dev libjack-jackd2-dev libmp3lame-dev libopencore-amrnb-dev libopencore-amrwb-dev librtmp-dev libsdl1.2-dev
+    apt-get -y install libtheora-dev libva-dev libvdpau-dev libvorbis-dev libx11-dev libxfixes-dev pkg-config texi2html yasm zlib1g-dev
+    apt-get -y install mercurial cvs subversion git-core libncurses-dev autoconf autopoint libtool automake  gettext nasm liba52-0.7.4-dev libpng12-dev libxt-dev libxv-dev libxvmc-dev libasound2-dev libmpcdec-dev libxvidcore-dev
+    clear
+    echo "                    Adesso Installo Libvpx 1.4.0"; sleep 4
+    cd /usr/local/src
+    rm -R libvpx
+    cp /home/`logname`/sifinstallrepo/libvpx-1.4.0.tar.bz2 /usr/local/src/;
+    tar xjf libvpx-1.4.0.tar.bz2; mv libvpx-1.4.0 libvpx;
+    rm -r libvpx-1.4.0.tar.bz2; cd libvpx
+    ./configure --enable-pic --enable-shared --enable-runtime_cpu_detect
+    make -j4
+    checkinstall --pkgname=libvpx --pkgversion="1.4.0-git" --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES;;
+
+
+
+
+#############################################################################################################################  
+#############################################################################################################################
+###                                                                                                                       ###
+##                                                                                                                         ##
+#                                                                                                                           #
+e1)                                       #########################################
+                                          #       Rimozione LIBVPX    1.4.0       #
+                                          #########################################
+
+
+    clear
+    echo "                    Adesso Rimuovo Libvpx 1.4.0"; sleep 4
+    apt-get -y purge libvpx; rm -r /usr/local/src/libvpx; rm /usr/local/src/packages/libvpx*;;
+
+
+
+
+#############################################################################################################################  
+#############################################################################################################################
+###                                                                                                                       ###
+##                                                                                                                         ##
+#                                                                                                                           # 
+f)                                        #########################################
+                                          #              FFmpeg 2.8.0             #
+                                          #########################################
+
+
+    apt-get -y remove ffmpeg
+    clear
+    apt-get update
+    ### Aggiunto Supporto per Linux Mint 16 Mate ###
+    apt-get -y install build-essential checkinstall git libfaac-dev libgpac-dev libjack-jackd2-dev libmp3lame-dev libopencore-amrnb-dev libopencore-amrwb-dev librtmp-dev libsdl1.2-dev
+    apt-get -y install libtheora-dev libva-dev libvdpau-dev libvorbis-dev libx11-dev libxfixes-dev pkg-config texi2html yasm zlib1g-dev
+    apt-get -y install mercurial cvs subversion git-core libncurses-dev autoconf autopoint libtool automake  gettext nasm liba52-0.7.4-dev libpng12-dev libxt-dev libxv-dev libxvmc-dev libasound2-dev libmpcdec-dev libxvidcore-dev
+    clear
+    apt-get -y install libass-dev libspeex-dev libmp3lame-dev libfaac-dev librtmp-dev libopencore-amrnb-dev libopencore-amrwb-dev  libiso9660-dev libvcdinfo-dev libjpeg-dev
+    clear
+    dialog --title "   ** Installazione FFmpeg 2.8.0 Git **   " --infobox "
+   ----------------------------------------------------
+          Seleziona il Tipo di Configurazione ...            
+   ----------------------------------------------------
+   
+   [a]  Installa FFmpeg con supporto X264 MPEG-4 AVC Encoder
+   
+   [b]  Installa FFmpeg con supporto X264 & X265 HEVC Encoder
+   
+   
+    
+      
+   ----------------------------------------------------" 0 0
+    echo -n "        Scegli  [a-b]:      "
+    read yourch
+    case $yourch in
+    a) clear
+    echo "                    Attendere qualche secondo la configurazione ...";
+    cd /usr/local/src;
+    rm -R ffmpeg
+    cp /home/`logname`/sifinstallrepo/ffmpeg-2.8.tar.bz2 /usr/local/src/;
+    tar xjf ffmpeg-2.8.tar.bz2; mv ffmpeg-2.8 ffmpeg;
+    rm -r ffmpeg-2.8.tar.bz2; cd ffmpeg
+    ./configure  --extra-libs="-ldl" --enable-gpl --enable-libass --enable-libfdk-aac --enable-libmp3lame --enable-libtheora --enable-libvorbis --disable-libvpx --enable-libx264 --enable-nonfree --enable-x11grab --enable-vdpau --enable-vaapi --enable-pic --enable-shared --disable-doc;
+    make -j4
+    checkinstall --pkgname=ffmpeg --pkgversion="7:2.8.0-git" --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES
+    hash -r
+    ldconfig -v
+    echo "                    Premere il tasto [Invio] per continuare. . .               "; read enterKey;;
+    b) clear
+    echo "                    Attendere qualche secondo la configurazione ...";
+    cd /usr/local/src;
+    rm -R ffmpeg
+    cp /home/`logname`/sifinstallrepo/ffmpeg-2.8.tar.bz2 /usr/local/src/;
+    tar xjf ffmpeg-2.8.tar.bz2; mv ffmpeg-2.8 ffmpeg;
+    rm -r ffmpeg-2.8.tar.bz2; cd ffmpeg 
+    ./configure  --extra-libs="-ldl" --enable-gpl --enable-libass --enable-libfdk-aac --enable-libmp3lame --enable-libtheora --enable-libvorbis --disable-libvpx --enable-libx264 --enable-libx265 --enable-nonfree --enable-x11grab --enable-vdpau --enable-vaapi --enable-pic --enable-shared --disable-doc;
+    make -j4
+    checkinstall --pkgname=ffmpeg --pkgversion="7:2.8.0-git" --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES
+    hash -r
+    ldconfig -v;;
+    esac
+    ;;
+
+
+
+
+#############################################################################################################################  
+#############################################################################################################################
+###                                                                                                                       ###
+##                                                                                                                         ##
+#                                                                                                                           #
+f1)                                       #########################################
+                                          #         Rimozione FFmpeg 2.8.0        #
+                                          #########################################
+
+
+    clear
+    echo "                    Adesso Rimuovo FFmpeg 2.8.0"; sleep 4
+    apt-get -y purge ffmpeg; rm -r /usr/local/src/ffmpeg; rm /usr/local/src/packages/ffmpeg*;; 
+    #esac
+    #echo "                    Premere il tasto [Invio] per continuare. . .               "; read enterKey;;
+
+
+
+
+#############################################################################################################################  
+#############################################################################################################################
+###                                                                                                                       ###
+##                                                                                                                         ##
+#                                                                                                                           #
+g)                                        #########################################
                                           #              LIBVPX 1.5.0             #
                                           #########################################
 
@@ -2139,15 +2299,15 @@
     make -j4
     checkinstall --pkgname=libvpx --pkgversion="1.5.0-git-snapshot" --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES;;
 
-    
-    
+
+
 
 #############################################################################################################################  
 #############################################################################################################################
 ###                                                                                                                       ###
 ##                                                                                                                         ##
 #                                                                                                                           #
-e1)                                       #########################################
+g1)                                       #########################################
                                           #             Rimozione LIBVPX          #
                                           #########################################
 
@@ -2156,16 +2316,16 @@
     echo "                    Adesso Rimuovo Libvpx 1.5.0"; sleep 4
     apt-get -y purge libvpx; rm -r /usr/local/src/libvpx; rm /usr/local/src/packages/libvpx*;;
 
-    
-    
+
+
 
 #############################################################################################################################  
 #############################################################################################################################
 ###                                                                                                                       ###
 ##                                                                                                                         ##
 #                                                                                                                           # 
-f)                                        #########################################
-                                          #                  FFMPEG               #
+h)                                        #########################################
+                                          #               FFmpeg 2.8.6            #
                                           #########################################
 
 
@@ -2177,7 +2337,7 @@
     apt-get -y install libtheora-dev libva-dev libvdpau-dev libvorbis-dev libx11-dev libxfixes-dev pkg-config texi2html yasm zlib1g-dev
     apt-get -y install mercurial cvs subversion git-core libncurses-dev autoconf autopoint libtool automake  gettext nasm liba52-0.7.4-dev libpng12-dev libxt-dev libxv-dev libxvmc-dev libasound2-dev libmpcdec-dev libxvidcore-dev
     clear
-    apt-get -y install libass-dev libspeex-dev libmp3lame-dev libfaac-dev librtmp-dev libopencore-amrnb-dev libopencore-amrwb-dev  libiso9660-dev libvcdinfo-dev libjpeg-dev
+    apt-get -y install libopus-dev libass-dev libspeex-dev libmp3lame-dev libfaac-dev librtmp-dev libopencore-amrnb-dev libopencore-amrwb-dev  libiso9660-dev libvcdinfo-dev libjpeg-dev
     clear
     dialog --title "   ** Installazione FFmpeg 2.8.6 Git-Snapshot **   " --infobox "
    ----------------------------------------------------
@@ -2217,25 +2377,24 @@
     tar xjf ffmpeg-2.8.6.tar.bz2; mv ffmpeg-2.8.6 ffmpeg;
     rm -r ffmpeg-2.8.6.tar.bz2; cd ffmpeg 
     #./configure  --extra-libs="-ldl" --enable-gpl --enable-libass --enable-libfdk-aac --enable-libmp3lame --enable-libtheora --enable-libvorbis --disable-libvpx --enable-libx264 --enable-libx265 --enable-nonfree --enable-x11grab --enable-vdpau --enable-vaapi --enable-pic --enable-shared --disable-doc;
-    ./configure  --extra-libs="-ldl" --enable-gpl --enable-libass --enable-libfdk-aac --enable-libfreetype --enable-libmp3lame  --enable-libtheora --enable-libvorbis --enable-libvpx --enable-libx264 --enable-libx265 --enable-nonfree --enable-nvenc --enable-libxvid --enable-x11grab --enable-vdpau --enable-vaapi --enable-pic --enable-shared --disable-doc
-    # --enable-libopus
+    ./configure  --extra-libs="-ldl" --enable-gpl --enable-libass --enable-libfdk-aac --enable-libfreetype --enable-libmp3lame --enable-libopus --enable-libtheora --enable-libvorbis --enable-libvpx --enable-libx264 --enable-libx265 --enable-nonfree --enable-nvenc --enable-libxvid --enable-x11grab --enable-vdpau --enable-vaapi --enable-pic --enable-shared --disable-doc
     make -j4
     checkinstall --pkgname=ffmpeg --pkgversion="7:2.8.6-git-snapshot" --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES
     hash -r;
     ldconfig -v;;
     esac
     ;;
-    
-    
-    
-    
+
+
+
+
 #############################################################################################################################  
 #############################################################################################################################
 ###                                                                                                                       ###
 ##                                                                                                                         ##
 #                                                                                                                           #
-f1)                                       #########################################
-                                          #             Rimozione FFMPEG          #
+h1)                                       #########################################
+                                          #          Rimozione FFmpeg 2.8.6       #
                                           #########################################
 
 
@@ -2244,9 +2403,9 @@
     apt-get -y purge ffmpeg; rm -r /usr/local/src/ffmpeg; rm /usr/local/src/packages/ffmpeg*;; 
     #esac
     #echo "                    Premere il tasto [Invio] per continuare. . .               "; read enterKey;;
-    
-  
-    
+
+
+
 
 #############################################################################################################################  
 #############################################################################################################################
@@ -3157,11 +3316,12 @@
         [e]    Trial 13, 16, 19°E               
         [f]    Quadri 13, 19, 23.5, 28.2°E      
         [g]    Quadri 9, 13, 16, 19°E           
-        [h]    Terrestre                        
-        [i]    Skip                             
+        [h]    Terrestre                  
+        
+        [z]    Skip                             
         
    ---------------------------------------------- " 0 0
-    echo -n "        Scegli  [a-i]:      "
+    echo -n "        Scegli  [a-z]:      "
     read yourch
     case $yourch in
 a) echo "$MONO13" >> /etc/vdr/diseqc.conf && cd /etc/vdr/ && wget http://www.vdr-italia.org/channels/17x/channels_tivusat_skyitalia_17x.conf && mv channels_tivusat_skyitalia_17x.conf channels.conf;;
@@ -4927,10 +5087,10 @@
         [i]    Rimozione Settings
         
         
-        [k]    Skip 
+        [z]    Skip 
         
    -------------------------------------------------------------- " 0 0
-    echo -n "        Scegli  [a-k]:      "
+    echo -n "        Scegli  [a-z]:      "
     read yourch
     case $yourch in
     
@@ -5639,13 +5799,13 @@
         *****  Installa  *****  ..........  *****  Rimuove  *****
         [a]    Installare KODI  ............  [a1]    Rimozione KODI  
         
-        [b]    Skip
+        [z]    Skip
         
         
 
            
    -------------------------------------------------------------- " 0 0
-    echo -n "        Scegli = Installa [a] ~~ Rimuove [a1] ~~ Esci [b]:      "
+    echo -n "        Scegli = Installa [a] ~~ Rimuove [a1] ~~ Esci [z]:      "
     read yourch
     case $yourch in
 
@@ -5750,10 +5910,11 @@
         [g]    Modifica oscam              
         [h]    Riconfigura lirc                        
         [i]    Nautilus  
-        [l]    Skip    
+        
+        [z]    Skip    
         
    ---------------------------------------------- " 0 0
-    echo -n "        Scegli  [a-l]:      " 
+    echo -n "        Scegli  [a-z]:      " 
     read yourch
     case $yourch in
 a) if cat /usr/local/bin/vdrrun | grep -q "xineliboutput";
@@ -5931,13 +6092,12 @@
     echo "                                                       .-----,        ,.----."
     echo "                                                .-----'       '------'       ',"
     echo "                                              ,'  First ed. by mona66 ® 2010   ',"
-    echo "                                              '  Updated by fiveten_59 ® 2013   '"
-    echo "                                              ',   Revision by knap ® 2013     ,'"
-    echo "                                              ,'   Updated by knap ® 2014      ',"
-    echo "                                              '  Updated by fiveten_59 ® 2015   '"
+    echo "                                              '     Updated and Revision        '"
+    echo "                                              ',  by fiveten_59 ® and knap ®    ,'"
+    echo "                                              ,'       2013 ~ 2014 ~ 2015      ',"
     echo "                                             '                                   '"
-    echo "                                              ',       Revision by knap         '"
-    echo "                                              ',     & fiveten_59 ® 2016       ,'"
+    echo "                                              ',     Revision by fiveten_59 ®   '"
+    echo "                                              ',          & knap ® ~ 2016     ,'"
     echo "                                                '----,      ,----,       ,----'"
     echo "                                                      '----'      '-----'"
     echo "                                                                O"
