--- installer	2016-06-20 22:14:42.000000000 +0200
+++ installer_5.7.0	2016-06-26 09:19:13.135053339 +0200
@@ -4,9 +4,9 @@
 # Nome Script : installer                                                               #
 # Versione: Github Repository                                                           #
 #                                                                                       #
-# Modificato per Kubuntu 13.04 ~ 15.04   32-64 Bit                                      #
-# Modificato per Debian 7.0   32-64 Bit                                                 #
-# Modificato per Linux Mint 15 ~ 16 Mate   32-64 Bit                                    #
+# Modificato per Kubuntu 14.04 ~ 16.04   32-64 Bit                                      #
+# Modificato per Debian 8.5   32-64 Bit                                                 #
+# Modificato per Linux Mint 15 ~ 17 Mate   32-64 Bit                                    #
 #                                                                                       #
 # Nuova Release Adattata per OS a 32 Bit                                                #
 # Compilazione Test eseguita con Vmware e Lubuntu 32 bit                                #                            
@@ -24,7 +24,7 @@
 # Scheda Video - NVIDIA GTX 750 Ti                                                      #
 # Scheda DVB-S - Skystar2 PCI   ver 2.3P                                                #
 #                                                                                       #
-# Driver Video - Nvidia 343.22 Installato - http://www.nvidia.it                        #
+# Driver Video - Nvidia 361.45 Installato - http://www.nvidia.it                        #
 #                                                                                       #
 # Per Configurazioni diverse questo Script puo' risultare non funzionante !             #
 #                                                                                       #
@@ -46,14 +46,14 @@
 # 
 PLUGIN1="--lirc -P'softhddevice -f' -P'dvbapi' -P'skindesigner -l /etc/vdr/logos/ -i /usr/local/share/vdr/plugins/skindesigner/skins/' -P'skinflatplus -l /etc/vdr/logos/' -P'tvguideng' -P'weatherforecast' -P'iptv' -P'tvguide -l /etc/vdr/logos/'  -P'text2skin' -P'osdteletext' -P'epgsearch' -P'externalplayer'"
 softhd_ico="[Desktop Entry]
-									Version=1.0
- 									Encoding=UTF-8
- 									Type=Application
- 									Name=Vdr-Softhddevice
-									Description=avvio di Vdr-Softhddevice
- 									Icon=/usr/local/src/icone/softhddevice.png
- 									Exec=/usr/local/bin/start-softhddevice-vdr.sh
- 									"
+Version=1.0
+Encoding=UTF-8
+Type=Application
+Name=Vdr-Softhddevice
+Description=avvio di Vdr-Softhddevice
+Icon=/usr/local/src/icone/softhddevice.png
+Exec=/usr/local/bin/start-softhddevice-vdr.sh
+"
 #
 Remind='è già installato... avete dimenticato di aggiungere il comando al vdrrun?'
 Remind1='è già installata... avete dimenticato di aggiungere il comando al vdrrun?'
@@ -337,9 +337,9 @@
     Nome Script: installer
     Versione: Github Repository
 
-    Modificato per Kubuntu 13.04 ~ 15.04   32-64 Bit      
-    Modificato per Debian 7.0   32-64 Bit                                         
-    Modificato per Linux Mint 15 ~ 16 Mate   32-64 Bit                            
+    Modificato per Kubuntu 14.04 ~ 16.04   32-64 Bit      
+    Modificato per Debian 8.5   32-64 Bit                                         
+    Modificato per Linux Mint 15 ~ 17 Mate   32-64 Bit                            
                                                                                       
                                                                                        
     Nuova Release Adattata per OS a 32 Bit                                                
@@ -358,7 +358,7 @@
     CPU - Intel Xeon E5-2658 V.2 - 2100Mhz - 8 core/16 thread -                           
     Scheda Video - NVIDIA GTX 750 Ti                                                      
     Scheda DVB-S - Skystar2 PCI   ver 2.3P                                                                                                                                     
-    Driver Video - Nvidia 343.22 Installato - http://www.nvidia.it                      
+    Driver Video - Nvidia 361.45 Installato - http://www.nvidia.it                      
                                                                                       
     Per Configurazioni diverse questo Script puo' risultare non funzionante !             
                                                                                       
@@ -385,7 +385,7 @@
           |   =======================================================================================   |
           |   ........♦♦♦   Librerie Di Supporto Audio Video & Codec - Installa & Rimuove  ♦♦♦........  |
           |   '''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''  |
-          |   [3]    +[Submenu] ~~  X264 git ~ X265 hg ~ Fdk-aac git ~ Libvpx git ~ FFmpeg git  .......  |         
+          |   [3]    +[Submenu] ~~  X264 Git ~ X265 hg ~ Fdk-aac Git ~ Libvpx Git ~ FFmpeg Git  .......  |         
           |   =======================================================================================   |
           |   .............♦♦♦   Installazione Xine-lib - Xine-ui - VDR - Plugins  - Skin ♦♦♦.........  |
           |   '''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''  |
@@ -469,32 +469,32 @@
     
 ############################### caricamento librerie ###################################
 
-if [ -s /home/`logname`/sifinstallrepo/yasm-1.3.0.tar.gz ]; then
+if [ -s /home/`logname`/sifinstallrepo/yasm ]; then
   echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦:::: ..... File Presente ..... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░";
 else
   echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦:::: ..... File non Presente ... Eseguo il Download ..... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░░░░░░░░░░";
-    wget https://www.dropbox.com/s/jtk3ax8hj7dmf2h/yasm-1.3.0.tar.gz;
-  fi
+    git clone https://github.com/yasm/yasm.git;
+  fi 
 
-if [ -s /home/`logname`/sifinstallrepo/libdrm-2.4.67.tar.gz ]; then
+if [ -s /home/`logname`/sifinstallrepo/libdrm-2.4.68.tar.gz ]; then
   echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦:::: ..... File Presente ..... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░";
 else
   echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦:::: ..... File non Presente ... Eseguo il Download ..... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░░░░░░░░░░";
-    wget https://dri.freedesktop.org/libdrm/libdrm-2.4.67.tar.gz;
+    wget https://dri.freedesktop.org/libdrm/libdrm-2.4.68.tar.gz;
   fi
 
-if [ -s /home/`logname`/sifinstallrepo/libva-1.7.0.tar.bz2 ]; then
+if [ -s /home/`logname`/sifinstallrepo/libva-1.7.1.tar.bz2 ]; then
   echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦:::: ..... File Presente ..... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░";
 else
   echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦:::: ..... File non Presente ... Eseguo il Download ..... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░░░░░░░░░░";
-    wget https://www.freedesktop.org/software/vaapi/releases/libva/libva-1.7.0.tar.bz2;
+    wget https://www.freedesktop.org/software/vaapi/releases/libva/libva-1.7.1.tar.bz2;
   fi
 
-if [ -s /home/`logname`/sifinstallrepo/libva-intel-driver-1.7.0.tar.bz2 ]; then
+if [ -s /home/`logname`/sifinstallrepo/libva-intel-driver-1.7.1.tar.bz2 ]; then
   echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦:::: ..... File Presente ..... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░";
 else
   echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦:::: ..... File non Presente ... Eseguo il Download ..... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░░░░░░░░░░";
-    wget https://www.freedesktop.org/software/vaapi/releases/libva-intel-driver/libva-intel-driver-1.7.0.tar.bz2;
+    wget https://www.freedesktop.org/software/vaapi/releases/libva-intel-driver/libva-intel-driver-1.7.1.tar.bz2;
   fi      
 
 if [ -s /home/`logname`/sifinstallrepo/libva-vdpau-driver-0.7.4.tar.bz2 ]; then
@@ -504,12 +504,12 @@
     wget http://www.freedesktop.org/software/vaapi/releases/libva-vdpau-driver/libva-vdpau-driver-0.7.4.tar.bz2;
   fi 
 
-if [ -s /home/`logname`/sifinstallrepo/libvdpau-1.1.1.tar.bz2 ]; then
+if [ -s /home/`logname`/sifinstallrepo/libvdpau ]; then
   echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦:::: ..... File Presente ..... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░";
 else
   echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦:::: ..... File non Presente ... Eseguo il Download ..... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░░░░░░░░░░";
-    wget https://people.freedesktop.org/~aplattner/vdpau/libvdpau-1.1.1.tar.bz2;
-  fi 
+    git clone https://github.com/aaronp24/libvdpau.git;
+fi
 
 if [ -s /home/`logname`/sifinstallrepo/vdpauinfo-1.0.tar.gz ]; then
   echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦:::: ..... File Presente ..... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░";
@@ -620,9 +620,6 @@
     wget https://www.dropbox.com/s/11clb1n2b6itave/patch_xineliboutput.tar.gz;
   fi
 
-# if [ -s /home/`logname`/sifinstallrepo/vdr-plugin-xine_0.9.4.orig.tar.gz ]; then
-#wget https://www.dropbox.com/s/bu8wm0a6gpw0tzx/vdr-plugin-xine_0.9.4.orig.tar.gz;
-
 if [ -s /home/`logname`/sifinstallrepo/vdr-plugin-xine ]; then
   echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦:::: ..... File Presente ..... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░";
 else
@@ -1012,7 +1009,6 @@
   echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦:::: ..... File Presente ..... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░";
 else
   echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦:::: ..... File non Presente ... Eseguo il Download ..... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░░░░░░░░░░";
-      #git clone https://github.com/maniacgit/vdr-play
       git clone http://projects.vdr-developer.org/git/vdr-plugin-play.git vdr-plugin-play;
 fi
 
@@ -1266,13 +1262,6 @@
     wget https://www.dropbox.com/s/tqua9jro2touw8c/vdr-getchannels-0.7.2.tar.gz;
   fi
 
-#if [ -s /home/`logname`/sifinstallrepo/channels-nohd-tivusat-skyno.conf ]; then
-#  echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦:::: ..... File Presente ..... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░";
-#else
-#  echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦:::: ..... File non Presente ... Eseguo il Download ..... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░░░░░░░░░░";
-#    wget http://www.vdr-italia.org/channels/17x/channels-nohd-tivusat-skyno.conf;
-#  fi
-
 if [ -s /home/`logname`/sifinstallrepo/channels.conf.terr.tar.gz ]; then
   echo "░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░¦¦¦¦¦¦:::: ..... File Presente ..... ::::¦¦¦¦¦¦░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░";
 else
@@ -1387,11 +1376,13 @@
 ######################################################################################  
 ############################ caricamento librerie ####################################
 
-    wget https://www.dropbox.com/s/jtk3ax8hj7dmf2h/yasm-1.3.0.tar.gz;
-    wget https://dri.freedesktop.org/libdrm/libdrm-2.4.67.tar.gz;
-    wget https://www.freedesktop.org/software/vaapi/releases/libva/libva-1.7.0.tar.bz2;
-    wget https://www.freedesktop.org/software/vaapi/releases/libva-intel-driver/libva-intel-driver-1.7.0.tar.bz2;
+    git clone https://github.com/yasm/yasm.git;
+    wget https://dri.freedesktop.org/libdrm/libdrm-2.4.68.tar.gz;
+    wget https://www.freedesktop.org/software/vaapi/releases/libva/libva-1.7.1.tar.bz2;
+    wget https://www.freedesktop.org/software/vaapi/releases/libva-intel-driver/libva-intel-driver-1.7.1.tar.bz2;
     wget http://www.freedesktop.org/software/vaapi/releases/libva-vdpau-driver/libva-vdpau-driver-0.7.4.tar.bz2;
+    git clone https://github.com/aaronp24/libvdpau.git;
+    wget https://people.freedesktop.org/~aplattner/vdpau/vdpauinfo-1.0.tar.gz;
     wget https://www.dropbox.com/s/3hbp5r2jf4a1lh8/vdpau_decode.patch;
     wget https://www.dropbox.com/s/u8h7u5htmqdzd0n/vdpau_dump.patch;
     wget https://www.dropbox.com/s/hvu4nstph14zaz5/utils_glx.patch;
@@ -1399,7 +1390,7 @@
     git clone git://git.videolan.org/x264.git;
     wget http://storage.googleapis.com/downloads.webmproject.org/releases/webm/libvpx-1.4.0.tar.bz2;
     wget http://storage.googleapis.com/downloads.webmproject.org/releases/webm/libvpx-1.5.0.tar.bz2;
-    git clone https://chromium.googlesource.com/webm/libvpx
+    git clone https://chromium.googlesource.com/webm/libvpx;
     wget https://github.com/FFmpeg/FFmpeg/releases/download/n2.8.6/ffmpeg-2.8.6.tar.bz2;
     wget https://github.com/FFmpeg/FFmpeg/releases/download/n3.0/ffmpeg-3.0.tar.bz2;
 
@@ -1431,7 +1422,6 @@
     git clone http://projects.vdr-developer.org/git/vdr-plugin-epgsearch.git;
     git clone git://github.com/manio/vdr-plugin-dvbapi.git;
     wget https://www.dropbox.com/s/5tyvqkbhohbbju2/dvbapi.patch.tar.gz;
-    #wget https://www.dropbox.com/s/bu8wm0a6gpw0tzx/vdr-plugin-xine_0.9.4.orig.tar.gz;
     git clone https://projects.vdr-developer.org/git/vdr-plugin-xine.git;
     wget https://www.dropbox.com/s/p4i5x8jdkfhirru/patch_Vdr_Xine.tar.gz;
     wget https://www.dropbox.com/s/ifjdxiq91mj3g3v/livepatch.tar.gz;
@@ -1506,7 +1496,6 @@
     wget https://www.dropbox.com/s/os0dz379o9rdtk3/getchannels.tar.gz;
     wget https://www.dropbox.com/s/tqua9jro2touw8c/vdr-getchannels-0.7.2.tar.gz;
     wget https://www.dropbox.com/s/jrizzen5q8o07uv/01_Add_language_It.patch.tar.gz;
-#    wget http://www.vdr-italia.org/channels/17x/channels-nohd-tivusat-skyno.conf;
     wget https://www.dropbox.com/s/z3czcyw5za82utc/channels.conf.dual.tar.gz;
     wget https://www.dropbox.com/s/reovn22nauzb2se/channels.conf.trial.tar.gz;
     wget https://www.dropbox.com/s/4jmjiurewriu1sh/channels.conf.terr.tar.gz;
@@ -1582,27 +1571,27 @@
    ---------------------------------------------------------------------------------------------
    
         ***** Installa  *****  .............................  *****  Rimuove  *****
-        [a]    YASM ................ 1.3.0  ................  [a1]    YASM ................ 1.3.0
+        [a]    YASM Git ............ 1.3.x  ................  [a1]    YASM Git ............ 1.3.x
         
-        [b]    LIBDRM .............. 2.4.67  ...............  [b1]    LIBDRM .............. 2.4.67
+        [b]    LIBDRM .............. 2.4.68  ...............  [b1]    LIBDRM .............. 2.4.68
         
-        [c]    LIBVA ............... 1.7.0  ................  [c1]    LIBVA ............... 1.7.0
-        [d]    LIBVA-INTEL-DRIVER .. 1.7.0  ................  [d1]    LIBVA-INTEL-DRIVER .. 1.7.0
+        [c]    LIBVA ............... 1.7.1  ................  [c1]    LIBVA ............... 1.7.1
+        [d]    LIBVA-INTEL-DRIVER .. 1.7.1  ................  [d1]    LIBVA-INTEL-DRIVER .. 1.7.1
         [e]    LIBVA-VDPAU-DRIVER .. 0.7.4  ................  [e1]    LIBVA-VDPAU-DRIVER .. 0.7.4
         
-        [f]    LIB-VDPAU ........... 1.1.1  ................  [f1]    LIB-VDPAU ........... 1.1.1
+        [f]    LIB-VDPAU Git ....... 1.1.x  ................  [f1]    LIB-VDPAU Git ....... 1.1.x
         [g]    VDPAUINFO ........... 1.0  ..................  [g1]    VDPAUINFO ........... 1.0
         
         [z]    Skip
         
    ---------------------------------------------------------------------------------------------      
-        - Note:  YASM ~ LIBDRM ~ LIBVA 1.7.0 - Vanno sempre installati come dipendenze.
+        - Note:  YASM ~ LIBDRM ~ LIBVA 1.7.1 - Vanno sempre installati come dipendenze.
         
         - Note:  LIBDRM ~ Mesa 3D and DRI [(Direct Rendering Infrastructure) by 3DFX, AMD (formerly ATI), Intel and Matrox]
         
         - Note:  LIBVA ~ Fornisce l'accesso all'accelerazione grafica hardware (GPU) GMA (Graphics Media Accelerator).
         
-        - Note:  LIBVA-INTEL-DRIVER 1.7.0 - is the VA-API implementation for Intel G45 chipsets
+        - Note:  LIBVA-INTEL-DRIVER 1.7.1 - is the VA-API implementation for Intel G45 chipsets
         and Intel HD Graphics for Intel Core processor family.
         Platform definitions:
         CTG: Cantiga, Intel GMA 4500MHD (GM45) - ILK: Ironlake, Intel HD Graphics for 2010 Intel Core processor family
@@ -1610,13 +1599,13 @@
         
         - Note:  LIBVA-VDPAU-DRIVER 0.7.4 - consists in a VDPAU-based backend for VA-API.
         
-        - Note:  LIB-VDPAU 1.1.1 - (Nvidia GPU hardware) Assegna parte del processo della decodifica e della 
+        - Note:  LIB-VDPAU 1.1.x - (Nvidia GPU hardware) Assegna parte del processo della decodifica e della 
         post-elaborazione video alla GPU della scheda video.
         
         - Note:
         Per chi vuole utilizzare libvdpau1 (LIB-VDPAU) installato dai Repository Ufficiali Ubuntu ...
         Non Deve Installare libvdpau1 (LIB-VDPAU) al Punto 2 / F - Nella Sezione dello Script-VDR 
-        il Pacchetto (LIB-VDPAU) libvdpau1 1.1.1 Viene Sviluppato e Distribuito da Repository Non Ufficiali
+        il Pacchetto (LIB-VDPAU) libvdpau1 1.1.x Viene Sviluppato e Distribuito da Repository Non Ufficiali
    --------------------------------------------------------------------------------------------- " 0 0
     echo -n "        Scegli = Installa [a-g] ~~ Rimuove [a1-g1] ~~ Esci [z]:      "
     read yourch
@@ -1631,22 +1620,20 @@
 ##                                                                                                                         ##
 #                                                                                                                           #
 a)                                        #########################################
-                                          #                   YASM                #
+                                          #              YASM  Github             #
                                           #########################################
 
 
     apt-get -y remove yasm
     clear
-    apt-get update && apt-get upgrade;
+    apt-get update && apt-get -y upgrade;
     apt-get install -y libpciaccess-dev checkinstall xserver-xorg-dev vainfo gcc build-essential pkg-config automake autoconf make autoproject;
     clear
-    echo "                    Adesso Installo Yasm 1.3.0"; sleep 4
+    echo "                    Adesso Installo Yasm Github"; sleep 4
     cd /usr/local/src; 
     rm -rf yasm;
-    cp /home/`logname`/sifinstallrepo/yasm-1.3.0.tar.gz /usr/local/src/;
-    tar -zxvf yasm-1.3.0.tar.gz; mv yasm-1.3.0 yasm; rm -r yasm-1.3.0.tar.gz; cd yasm
-    ./configure
-    checkinstall --pkgname=yasm --pkgversion="1.3.0-git" --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES;;
+    cp -Rf /home/`logname`/sifinstallrepo/yasm /usr/local/src/; cd yasm; ./autogen.sh ; ./configure;
+    checkinstall --pkgname=yasm --pkgversion="1.3.x-Github" --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES;;
     
     
     
@@ -1662,7 +1649,7 @@
 
 
     clear
-    echo "                    Adesso Rimuovo Yasm 1.3.0"; sleep 4
+    echo "                    Adesso Rimuovo Yasm Github"; sleep 4
     apt-get -y purge yasm; rm -r /usr/local/src/yasm; rm /usr/local/src/packages/yasm*;;
     
     
@@ -1674,20 +1661,20 @@
 ##                                                                                                                         ##
 #                                                                                                                           #
 b)                                        #########################################
-                                          #                  LIBDRM               #
+                                          #                 LIBDRM                #
                                           #########################################
 
 
     apt-get -y remove libdrm
     clear
-    echo "                    Adesso Installo Libdrm 2.4.67"; sleep 4
+    echo "                    Adesso Installo Libdrm 2.4.68"; sleep 4
     apt-get install -y libpthread-stubs0-dev libpciaccess-dev
     cd /usr/local/src
     rm -rf libdrm;
-    cp /home/`logname`/sifinstallrepo/libdrm-2.4.67.tar.gz /usr/local/src/;  
-    tar -zxvf libdrm-2.4.67.tar.gz; mv libdrm-2.4.67 libdrm; rm -r libdrm-2.4.67.tar.gz; cd libdrm
+    cp /home/`logname`/sifinstallrepo/libdrm-2.4.68.tar.gz /usr/local/src/;  
+    tar -zxvf libdrm-2.4.68.tar.gz; mv libdrm-2.4.68 libdrm; rm -r libdrm-2.4.68.tar.gz; cd libdrm
     ./configure --enable-silent-rules --enable-libkms --enable-udev --enable-largefile --enable-intel --enable-radeon
-    checkinstall --pkgname=libdrm --pkgversion="2.4.67-git" --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES;;
+    checkinstall --pkgname=libdrm --pkgversion="2.4.68-Git" --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES;;
     
     
     
@@ -1703,7 +1690,7 @@
 
 
     clear
-    echo "                    Adesso Rimuovo Libdrm 2.4.67"; sleep 4
+    echo "                    Adesso Rimuovo Libdrm 2.4.68"; sleep 4
     apt-get -y purge libdrm; rm -r /usr/local/src/libdrm; rm /usr/local/src/packages/libdrm*;;
     
     
@@ -1722,14 +1709,14 @@
     apt-get -y remove libva
     apt-get -y install libwayland-dev
     clear
-    echo "                    Adesso Installo Libva 1.7.0"; sleep 4
+    echo "                    Adesso Installo Libva 1.7.1"; sleep 4
     cd /usr/local/src;
     rm -r libva;
-    cp /home/`logname`/sifinstallrepo/libva-1.7.0.tar.bz2 /usr/local/src/;
-    tar xjf libva-1.7.0.tar.bz2; mv libva-1.7.0 libva; rm -r libva-1.7.0.tar.bz2;
+    cp /home/`logname`/sifinstallrepo/libva-1.7.1.tar.bz2 /usr/local/src/;
+    tar xjf libva-1.7.1.tar.bz2; mv libva-1.7.1 libva; rm -r libva-1.7.1.tar.bz2;
     cd libva
     ./configure
-    checkinstall --pkgname=libva --pkgversion="1.7.0-git" --pkgrelease=VAAPI-0.38.0 --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES
+    checkinstall --pkgname=libva --pkgversion="1.7.1-Git" --pkgrelease=VAAPI-0.38.0 --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES
     rm /etc/ld.so.conf;
     touch /etc/ld.so.conf;
     echo 'include /etc/ld.so.conf.d/*.conf
@@ -1750,7 +1737,7 @@
 
 
     clear
-    echo "                    Adesso Rimuovo Libva 1.7.0"; sleep 4
+    echo "                    Adesso Rimuovo Libva 1.7.1"; sleep 4
     apt-get -y purge libva; rm -r /usr/local/src/libva; rm /usr/local/src/packages/libva*;;
     
     
@@ -1762,21 +1749,21 @@
 ##                                                                                                                         ##
 #                                                                                                                           #
 d)                                        ##########################################
-                                          #        LIBVA-INTEL-DRIVER 1.7.0        #
+                                          #        LIBVA-INTEL-DRIVER 1.7.1        #
                                           ##########################################
 
 
     apt-get -y remove libva-intel-driver
     apt-get -y install libwayland-dev
     clear
-    echo "                    Adesso Installo libva-intel-driver 1.7.0"; sleep 4
+    echo "                    Adesso Installo libva-intel-driver 1.7.1"; sleep 4
     cd /usr/local/src;
     rm -r libva-intel-driver;
-    cp /home/`logname`/sifinstallrepo/libva-intel-driver-1.7.0.tar.bz2 /usr/local/src/;
-    tar xjf libva-intel-driver-1.7.0.tar.bz2; mv libva-intel-driver-1.7.0 libva-intel-driver; rm -r libva-intel-driver-1.7.0.tar.bz2;
+    cp /home/`logname`/sifinstallrepo/libva-intel-driver-1.7.1.tar.bz2 /usr/local/src/;
+    tar xjf libva-intel-driver-1.7.1.tar.bz2; mv libva-intel-driver-1.7.1 libva-intel-driver; rm -r libva-intel-driver-1.7.1.tar.bz2;
     cd libva-intel-driver
     ./configure
-    checkinstall --pkgname=libva-intel-driver --pkgversion="1.7.0-git" --pkgrelease=VAAPI-0.38.0 --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES
+    checkinstall --pkgname=libva-intel-driver --pkgversion="1.7.1-Git" --pkgrelease=VAAPI-0.38.0 --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES
     rm /etc/ld.so.conf;
     touch /etc/ld.so.conf;
     echo 'include /etc/ld.so.conf.d/*.conf
@@ -1792,12 +1779,12 @@
 ##                                                                                                                         ##
 #                                                                                                                           #
 d1)                                       ##########################################
-                                          #   Rimozione LIBVA-INTEL-DRIVER 1.7.0   #
+                                          #   Rimozione LIBVA-INTEL-DRIVER 1.7.1   #
                                           ##########################################
 
 
     clear
-    echo "                    Adesso Rimuovo libva-intel-driver 1.7.0"; sleep 4
+    echo "                    Adesso Rimuovo libva-intel-driver 1.7.1"; sleep 4
     apt-get -y purge libva; rm -r /usr/local/src/libva; rm /usr/local/src/packages/libva*;;
     
     
@@ -1830,7 +1817,7 @@
     patch -p0 < utils_glx.patch; sleep 1;
     cd $SRC/libva-vdpau-driver
     ./configure
-    checkinstall --pkgname=libva-vdpau-driver --pkgversion="0.7.4-git" --pkgrelease=VAAPI-0.38.0 --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES
+    checkinstall --pkgname=libva-vdpau-driver --pkgversion="0.7.4-Git" --pkgrelease=VAAPI-0.38.0 --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES
     rm /etc/ld.so.conf;
     touch /etc/ld.so.conf;
     echo 'include /etc/ld.so.conf.d/*.conf
@@ -1863,20 +1850,18 @@
 ##                                                                                                                         ##
 #                                                                                                                           #
 f)                                        ##########################################
-                                          #             LIB-VDPAU 1.1.1            #
+                                          #         LIB-VDPAU 1.1.x Github         #
                                           ##########################################
 
 
     apt-get -y remove libvdpau1
     apt-get -y install libxext-dev
     clear
-    echo "                    Adesso Installo libvdpau 1.1.1"; sleep 4
+    echo "                    Adesso Installo libvdpau Github"; sleep 4
     cd /usr/local/src;
     rm -r libvdpau;
-    cp /home/`logname`/sifinstallrepo/libvdpau-1.1.1.tar.bz2 /usr/local/src/;
-    tar xjf libvdpau-1.1.1.tar.bz2; mv libvdpau-1.1.1 libvdpau; rm -r libvdpau-1.1.1.tar.bz2; cd $SRC/libvdpau/;
-    ./configure
-    checkinstall --pkgname=libvdpau1 --pkgversion="1.1.1" --pkgrelease=vdpau --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES
+    cp -Rf /home/`logname`/sifinstallrepo/libvdpau /usr/local/src/; cd $SRC/libvdpau/; ./autogen.sh ; ./configure;
+    checkinstall --pkgname=libvdpau1 --pkgversion="1.1.x-Git" --pkgrelease=vdpau --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES
     rm /etc/ld.so.conf;
     touch /etc/ld.so.conf;
     echo 'include /etc/ld.so.conf.d/*.conf
@@ -1892,12 +1877,12 @@
 ##                                                                                                                         ##
 #                                                                                                                           #
 f1)                                       ##########################################
-                                          #       Rimozione   LIB-VDPAU 1.1.1      #
+                                          #    Rimozione LIB-VDPAU 1.1.x Github    #
                                           ##########################################
 
 
     clear
-    echo "                    Adesso Rimuovo libvdpau 1.1.1"; sleep 4
+    echo "                    Adesso Rimuovo libvdpau 1.1.x-Git"; sleep 4
     apt-get -y purge libvdpau1; rm -r /usr/local/src/libvdpau; rm /usr/local/src/packages/libvdpau1*;;
 
 
@@ -1909,7 +1894,7 @@
 ##                                                                                                                         ##
 #                                                                                                                           #
 g)                                        ##########################################
-                                          #               VDPAUINFO 1.0            #
+                                          #              VDPAUINFO 1.0             #
                                           ##########################################
 
 
@@ -1971,7 +1956,7 @@
    --------------------------------------------------------------------------------------------------------------------
         
         *****  Installa  *****  ...........................  *****  Rimuove  *****
-        [a]    X264 26.09.2015 Git  .......................  [a1]    X264 26.09.2015 Git
+        [a]    X264 Git  ..................................  [a1]    X264 Git
         [b]    X265 HG  ...................................  [b1]    X265 HG
         [c]    FDK-AAC Git  ...............................  [c1]    FDK-AAC Git
         [d]    NVIDIA Video Codec 6.1 SDK .................  [d1]    NVIDIA Video Codec 6.1 SDK
@@ -2015,13 +2000,13 @@
 ##                                                                                                                         ##
 #                                                                                                                           #
 a)                                        #########################################
-                                          #                   X264                #
+                                          #               X264  Github            #
                                           #########################################
 
 
     apt-get -y remove x264
     clear
-    apt-get update;
+    apt-get -y update;
     apt-get -y install build-essential checkinstall git libfaac-dev libgpac-dev libjack-jackd2-dev libmp3lame-dev libopencore-amrnb-dev libopencore-amrwb-dev librtmp-dev libsdl1.2-dev
     #apt-get -y remove libvdpau1 libvdpau-dev;     apt-get -y auto-remove libvdpau1 libvdpau-dev;
     #apt-get -y install libvdpau1 libvdpau-dev
@@ -2036,7 +2021,7 @@
     cd x264; git checkout stable;
     ./configure --enable-pic --enable-shared --extra-cflags="-mtune=native -march=native -mfpmath=sse -O4 -pipe"
     make -j4
-    checkinstall --pkgname=x264 --pkgversion="2:26.09.2015-git" --bk --default --deldoc=yes --pakdir=$PACKAGES
+    checkinstall --pkgname=x264 --pkgversion="2:0.143.0000-Git" --bk --default --deldoc=yes --pakdir=$PACKAGES
     ldconfig -v;;  
 
     
@@ -2065,7 +2050,7 @@
 ##                                                                                                                         ##
 #                                                                                                                           #
 b)                                        #########################################
-                                          #                  X265                 #
+                                          #               X265  Github            #
                                           #########################################
 
 
@@ -2079,7 +2064,7 @@
     cd x265/build/linux;
     ./make-Makefiles.bash
     make -j4
-    checkinstall --pkgname=x265 --pkgversion="2:1.7-git" --bk --default --deldoc=yes --pakdir=$PACKAGES;
+    checkinstall --pkgname=x265 --pkgversion="2:1.7-Git" --bk --default --deldoc=yes --pakdir=$PACKAGES;
     ldconfig -v;;
 
     
@@ -2108,7 +2093,7 @@
 ##                                                                                                                         ##
 #                                                                                                                           #
 c)                                        #########################################
-                                          #                 FDK-AAC               #
+                                          #             FDK-AAC  Github           #
                                           #########################################
 
 
@@ -2121,7 +2106,7 @@
     autoreconf -fiv
     ./configure --disable-shared --with-pic
     make -j4
-    checkinstall --pkgname=fdk-aac --pkgversion="0.1.x-git" --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES;;
+    checkinstall --pkgname=fdk-aac --pkgversion="0.1.x-Git" --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES;;
 
     
     
@@ -2212,7 +2197,7 @@
     cd libvpx
     ./configure --enable-pic --enable-shared --enable-runtime_cpu_detect
     make -j4
-    checkinstall --pkgname=libvpx --pkgversion="1.6.0-git" --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES;;
+    checkinstall --pkgname=libvpx --pkgversion="1.6.0-Git" --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES;;
 
 
 
@@ -2284,7 +2269,7 @@
     #./configure  --extra-libs="-ldl" --enable-gpl --enable-libass --enable-libfdk-aac --enable-libmp3lame --enable-libtheora --enable-libvorbis --disable-libvpx --enable-libx264 --enable-nonfree --enable-x11grab --enable-vdpau --enable-vaapi --enable-pic --enable-shared --disable-doc;
     ./configure  --extra-libs="-ldl" --enable-gpl --enable-libass --enable-libfdk-aac --enable-libfreetype --enable-libmp3lame --enable-libopus --enable-libtheora --enable-libvorbis --enable-libvpx --enable-libx264 --enable-nonfree --enable-nvenc --enable-libxvid --enable-x11grab --enable-vdpau --enable-vaapi --enable-pic --enable-shared --disable-doc
     make -j4
-    checkinstall --pkgname=ffmpeg --pkgversion="7:2.8.6-git-snapshot" --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES
+    checkinstall --pkgname=ffmpeg --pkgversion="7:2.8.6-Git-snapshot" --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES
     hash -r
     ldconfig -v
     echo "                    Premere il tasto [Invio] per continuare. . .               "; read enterKey;;
@@ -2298,7 +2283,7 @@
     #./configure  --extra-libs="-ldl" --enable-gpl --enable-libass --enable-libfdk-aac --enable-libmp3lame --enable-libtheora --enable-libvorbis --disable-libvpx --enable-libx264 --enable-libx265 --enable-nonfree --enable-x11grab --enable-vdpau --enable-vaapi --enable-pic --enable-shared --disable-doc;
     ./configure  --extra-libs="-ldl" --enable-gpl --enable-libass --enable-libfdk-aac --enable-libfreetype --enable-libmp3lame --enable-libopus --enable-libtheora --enable-libvorbis --enable-libvpx --enable-libx264 --enable-libx265 --enable-nonfree --enable-nvenc --enable-libxvid --enable-x11grab --enable-vdpau --enable-vaapi --enable-pic --enable-shared --disable-doc
     make -j4
-    checkinstall --pkgname=ffmpeg --pkgversion="7:2.8.6-git-snapshot" --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES
+    checkinstall --pkgname=ffmpeg --pkgversion="7:2.8.6-Git-snapshot" --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES
     hash -r;
     ldconfig -v;;
     esac
@@ -2377,7 +2362,7 @@
     ./configure  --extra-libs="-ldl" --enable-gpl --enable-libass --enable-libfdk-aac --enable-libfreetype --enable-libmp3lame --enable-libopus --enable-libtheora --enable-libvorbis --enable-libvpx --enable-libx264 --enable-nonfree --enable-nvenc --enable-libxvid --enable-x11grab --enable-vdpau --enable-vaapi --enable-pic --enable-shared --disable-doc
     echo "                    Premere il tasto [Invio] per continuare. . .               "; read enterKey;
     make -j4
-    checkinstall --pkgname=ffmpeg --pkgversion="7:3.0-git" --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES
+    checkinstall --pkgname=ffmpeg --pkgversion="7:3.0-Git" --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES
     hash -r
     ldconfig -v
     echo "                    Premere il tasto [Invio] per continuare. . .               "; read enterKey;;
@@ -2390,7 +2375,7 @@
     rm -r ffmpeg-3.0.tar.bz2; cd ffmpeg 
     ./configure  --extra-libs="-ldl" --enable-gpl --enable-libass --enable-libfdk-aac --enable-libfreetype --enable-libmp3lame --enable-libopus --enable-libtheora --enable-libvorbis --enable-libvpx --enable-libx264 --enable-libx265 --enable-nonfree --enable-nvenc --enable-libxvid --enable-x11grab --enable-vdpau --enable-vaapi --enable-pic --enable-shared --disable-doc
     make -j4
-    checkinstall --pkgname=ffmpeg --pkgversion="7:3.0-git" --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES
+    checkinstall --pkgname=ffmpeg --pkgversion="7:3.0-Git" --backup=no --deldoc=yes --fstrans=no --default --pakdir=$PACKAGES
     hash -r
     ldconfig -v;;
     esac
@@ -2495,7 +2480,7 @@
     sh ./autogen.sh;
     ./configure --prefix=/usr --disable-static;
     make;
-    checkinstall --pkgname=libass --pkgversion="0.13.2-git" --backup=no --deldoc=yes --fstrans=no --default;
+    checkinstall --pkgname=libass --pkgversion="0.13.2-Git" --backup=no --deldoc=yes --fstrans=no --default;
     #
     ln -s /usr/lib/libass.so /usr/lib/libass.so.4;;
 
@@ -2513,7 +2498,7 @@
     
     
     clear
-    echo "                    Adesso Rimuovo Libass 0.13.2-git"; sleep 4
+    echo "                    Adesso Rimuovo Libass 0.13.2-Git"; sleep 4
     apt-get -y purge libass; rm -r /usr/local/src/libass*; rm /usr/local/src/packages/libass*;
     echo "                    Premere il tasto [Invio] per continuare. . .               "; read enterKey;;
 
@@ -2538,9 +2523,8 @@
     ./bootstrap.py;
     ./waf configure --prefix=/usr/;
     ./waf build -p --disable-debug-build --enable-libmpv-shared;
-    ./waf install;;
-    
-    
+    ./waf install;
+    ldconfig -v;;    
 
 
 #############################################################################################################################  
@@ -2603,7 +2587,7 @@
     export LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH
     ./autogen.sh --disable-dxr3 --enable-w32dll --with-pic --with-w32-path=/usr/local/lib/win32
     make -j4 
-    checkinstall --fstrans=no --install=yes --pkgname=libxine2 --pkgversion "1.2.6-git" --pakdir=$PACKAGES --default
+    checkinstall --fstrans=no --install=yes --pkgname=libxine2 --pkgversion "1.2.6-Git" --pakdir=$PACKAGES --default
     echo "                    Premere il tasto [Invio] per continuare. . .               "; read enterKey;;
 
     
@@ -2650,7 +2634,7 @@
     tar -xJf  xine-ui-0.99.9.tar.xz; mv xine-ui-0.99.9 xine-ui; rm -r xine-ui-0.99.9.tar.xz; cd xine-ui;
     ./autogen.sh --enable-vdr-keys
     make -j4
-    checkinstall --fstrans=no --install=yes --pkgname=xine-ui --pkgversion "0.99.9-git" --default
+    checkinstall --fstrans=no --install=yes --pkgname=xine-ui --pkgversion "0.99.9-Git" --default
     ldconfig
     fi
     echo "                    Premere il tasto [Invio] per continuare. . .               "; read enterKey;;
@@ -2769,7 +2753,7 @@
     echo ""; echo "*** Final compilation ***"; echo ""
     cd /usr/local/src/vdr
     apt-get -y purge vdr
-    checkinstall --fstrans=no --install=yes --pkgname=vdr --pkgversion "2.2.0-git" --pakdir=$PACKAGES --default
+    checkinstall --fstrans=no --install=yes --pkgname=vdr --pkgversion "2.2.0-Git" --pakdir=$PACKAGES --default
     rm -r /media/video; mkdir /media/video; mkdir -p /usr/local/lib/vdr/scripts;
     mkdir -p /etc/vdr/plugins; cp *.conf /etc/vdr; mkdir -p /usr/local/lib/vdr/plugins;
     clear    
